
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Documentation for the centrally provided Research Computing HPC services at UCL.">
      
      
        <meta name="author" content="UCL RCAS Team">
      
      
        <link rel="canonical" href="https://www.rc.ucl.ac.uk/docs/Installed_Software_Lists/r-packages/">
      
      
        <link rel="prev" href="../python-packages/">
      
      
        <link rel="next" href="../../Other_Services/Aristotle/">
      
      
      <link rel="icon" href="../../img/favicon.ico">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.11">
    
    
      
        <title>R Packages - UCL Research Computing Documentation</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.4af4bdda.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
    
    
      <link rel="stylesheet" href="../../stylesheets/tweaks.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="deep-purple" data-md-color-accent="deep-purple">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#r-packages" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="UCL Research Computing Documentation" class="md-header__button md-logo" aria-label="UCL Research Computing Documentation" data-md-component="logo">
      
  <img src="../../img/portico.svg" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            UCL Research Computing Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              R Packages
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="deep-purple" data-md-color-accent="deep-purple"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3m0-7 2.39 3.42C13.65 5.15 12.84 5 12 5s-1.65.15-2.39.42zM3.34 7l4.16-.35A7.2 7.2 0 0 0 5.94 8.5c-.44.74-.69 1.5-.83 2.29zm.02 10 1.76-3.77a7.131 7.131 0 0 0 2.38 4.14zM20.65 7l-1.77 3.79a7.02 7.02 0 0 0-2.38-4.15zm-.01 10-4.14.36c.59-.51 1.12-1.14 1.54-1.86.42-.73.69-1.5.83-2.29zM12 22l-2.41-3.44c.74.27 1.55.44 2.41.44.82 0 1.63-.17 2.37-.44z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="deep-purple" data-md-color-accent="deep-purple"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m17.75 4.09-2.53 1.94.91 3.06-2.63-1.81-2.63 1.81.91-3.06-2.53-1.94L12.44 4l1.06-3 1.06 3zm3.5 6.91-1.64 1.25.59 1.98-1.7-1.17-1.7 1.17.59-1.98L15.75 11l2.06-.05L18.5 9l.69 1.95zm-2.28 4.95c.83-.08 1.72 1.1 1.19 1.85-.32.45-.66.87-1.08 1.27C15.17 23 8.84 23 4.94 19.07c-3.91-3.9-3.91-10.24 0-14.14.4-.4.82-.76 1.27-1.08.75-.53 1.93.36 1.85 1.19-.27 2.86.69 5.83 2.89 8.02a9.96 9.96 0 0 0 8.02 2.89m-1.64 2.02a12.08 12.08 0 0 1-7.8-3.47c-2.17-2.19-3.33-5-3.49-7.82-2.81 3.14-2.7 7.96.31 10.98 3.02 3.01 7.84 3.12 10.98.31"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/UCL-ARC/mkdocs-rc-docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6m-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3m44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9M244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8M97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1m-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7m32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1m-11.4-14.7c-1.6 1-1.6 3.6 0 5.9s4.3 3.3 5.6 2.3c1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2"/></svg>
  </div>
  <div class="md-source__repository">
    UCL-ARC/mkdocs-rc-docs
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="UCL Research Computing Documentation" class="md-nav__button md-logo" aria-label="UCL Research Computing Documentation" data-md-component="logo">
      
  <img src="../../img/portico.svg" alt="logo">

    </a>
    UCL Research Computing Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/UCL-ARC/mkdocs-rc-docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6m-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3m44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9M244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8M97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1m-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7m32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1m-11.4-14.7c-1.6 1-1.6 3.6 0 5.9s4.3 3.3 5.6 2.3c1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2"/></svg>
  </div>
  <div class="md-source__repository">
    UCL-ARC/mkdocs-rc-docs
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Research Computing Services
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Account_Services/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Account Services
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Additional_Resource_Requests/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Additional Resource Requests
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Contact_Us/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Contact and Support
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Data_Management/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Management
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Example_Jobscripts/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Example Jobscripts
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Experienced_Users/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Quick Start Guide for Experienced HPC Users
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Interactive_Jobs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Interactive Job Sessions
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Job_Results/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Where do my results go?
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../New_Users/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Guide for New Users
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Planned_Outages/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Planned Outages
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Remote_Access/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Remote Access
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Status_page/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Status of machines
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Terms_and_Conditions/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Terms and Conditions
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../UCL_Service_For_Me/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    UCL Service For Me
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../howto/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    How To
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_17" >
        
          
          <label class="md-nav__link" for="__nav_17" id="__nav_17_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Background
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_17_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_17">
            <span class="md-nav__icon md-icon"></span>
            Background
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Background/Cluster_Computing/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Cluster Computing
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Background/Data_Storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Storage
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Background/Parallel_Filesystems/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Parallel Filesystems
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_18" >
        
          
          <label class="md-nav__link" for="__nav_18" id="__nav_18_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Clusters
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_18_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_18">
            <span class="md-nav__icon md-icon"></span>
            Clusters
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Clusters/Myriad/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Myriad
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Clusters/Kathleen/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Kathleen
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Clusters/Michael/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MMM Michael
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Clusters/Young/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MMM Hub Young
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Clusters/Acknowledging_RC_Systems/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Acknowledging the Use of RC Systems
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_19" checked>
        
          
          <label class="md-nav__link" for="__nav_19" id="__nav_19_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Installed Software Lists
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_19_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_19">
            <span class="md-nav__icon md-icon"></span>
            Installed Software Lists
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../module-packages/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    General Software Lists
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../python-packages/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Python Packages
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    R Packages
    
  </span>
  

      </a>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_20" >
        
          
          <label class="md-nav__link" for="__nav_20" id="__nav_20_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Other Services
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_20_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_20">
            <span class="md-nav__icon md-icon"></span>
            Other Services
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Other_Services/Aristotle/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Aristotle
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Other_Services/JupyterHub/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    JupyterHub
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Other_Services/RStudio/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    RStudio
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Other_Services/UCL_UK_e-Science_Certificates/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    UCL UK e-Science Certificates
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_21" >
        
          
          <label class="md-nav__link" for="__nav_21" id="__nav_21_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Paid-For Resources
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_21_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_21">
            <span class="md-nav__icon md-icon"></span>
            Paid-For Resources
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Paid-For_Resources/How_to_Use/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    How to Use
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Paid-For_Resources/Purchasing_in_Myriad/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Purchasing in Myriad
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_22" >
        
          
          <label class="md-nav__link" for="__nav_22" id="__nav_22_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Software Guides
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_22_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_22">
            <span class="md-nav__icon md-icon"></span>
            Software Guides
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Software_Guides/ANSYS/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    ANSYS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Software_Guides/Matlab/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MATLAB
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Software_Guides/Singularity/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Singularity/Apptainer
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Software_Guides/R/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    R
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Software_Guides/Other_Software/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Other Software
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Software_Guides/AlphaFold3/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    AlphaFold3
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Software_Guides/Installing_Software/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Installing Software
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Software_Guides/RFAA/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    RoseTTAFold All-Atom
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_23" >
        
          
          <label class="md-nav__link" for="__nav_23" id="__nav_23_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Walkthroughs
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_23_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_23">
            <span class="md-nav__icon md-icon"></span>
            Walkthroughs
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Walkthroughs/Giving_Files/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Giving Files to Another User
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Walkthroughs/Logging_In/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Logging In
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../Walkthroughs/Machine_Learning/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Machine Learning
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="r-packages">R Packages<a class="headerlink" href="#r-packages" title="Permanent link">§</a></h1>
<p>We provide a collection of installed R packages for each release of R, as a <a href="../module-packages/">bundle module</a>. This page lists the packages for the current <code>recommended</code> R bundle.</p>
<p>This can be loaded using:</p>
<div class="codehilite"><pre><span></span><code>module load r/recommended
</code></pre></div>

<p>The version of R provided with this bundle is currently R version 4.2.0 (2022-04-22).</p>
<p>The following list was last updated at: 17:05:17 (+0100) on 04 Apr 2025.</p>
<table>
<thead>
<tr>
<th style="text-align: center;">Module</th>
<th style="text-align: center;">Version</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: center;"><code>abc</code></td>
<td style="text-align: center;">2.2.1</td>
<td style="text-align: left;">Tools for Approximate Bayesian Computation (ABC)</td>
</tr>
<tr>
<td style="text-align: center;"><code>abc.data</code></td>
<td style="text-align: center;">1.0</td>
<td style="text-align: left;">Data Only: Tools for Approximate Bayesian Computation (ABC)</td>
</tr>
<tr>
<td style="text-align: center;"><code>abind</code></td>
<td style="text-align: center;">1.4-5</td>
<td style="text-align: left;">Combine Multidimensional Arrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>ade4</code></td>
<td style="text-align: center;">1.7-19</td>
<td style="text-align: left;">Analysis of Ecological Data: Exploratory and Euclidean Methods in Environmental Sciences</td>
</tr>
<tr>
<td style="text-align: center;"><code>adegenet</code></td>
<td style="text-align: center;">2.1.6</td>
<td style="text-align: left;">Exploratory Analysis of Genetic and Genomic Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>ADGofTest</code></td>
<td style="text-align: center;">0.3</td>
<td style="text-align: left;">Anderson-Darling GoF test</td>
</tr>
<tr>
<td style="text-align: center;"><code>admisc</code></td>
<td style="text-align: center;">0.27</td>
<td style="text-align: left;">Adrian Dusa's Miscellaneous</td>
</tr>
<tr>
<td style="text-align: center;"><code>afex</code></td>
<td style="text-align: center;">1.2-1</td>
<td style="text-align: left;">Analysis of Factorial Experiments</td>
</tr>
<tr>
<td style="text-align: center;"><code>affxparser</code></td>
<td style="text-align: center;">1.68.1</td>
<td style="text-align: left;">Affymetrix File Parsing SDK</td>
</tr>
<tr>
<td style="text-align: center;"><code>affy</code></td>
<td style="text-align: center;">1.74.0</td>
<td style="text-align: left;">Methods for Affymetrix Oligonucleotide Arrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>affydata</code></td>
<td style="text-align: center;">1.44.0</td>
<td style="text-align: left;">Affymetrix Data for Demonstration Purpose</td>
</tr>
<tr>
<td style="text-align: center;"><code>affyio</code></td>
<td style="text-align: center;">1.66.0</td>
<td style="text-align: left;">Tools for parsing Affymetrix data files</td>
</tr>
<tr>
<td style="text-align: center;"><code>affylmGUI</code></td>
<td style="text-align: center;">1.70.0</td>
<td style="text-align: left;">GUI for limma Package with Affymetrix Microarrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>affyPLM</code></td>
<td style="text-align: center;">1.72.0</td>
<td style="text-align: left;">Methods for fitting probe-level models</td>
</tr>
<tr>
<td style="text-align: center;"><code>akima</code></td>
<td style="text-align: center;">0.6-3.4</td>
<td style="text-align: left;">Interpolation of Irregularly and Regularly Spaced Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>annaffy</code></td>
<td style="text-align: center;">1.68.0</td>
<td style="text-align: left;">Annotation tools for Affymetrix biological metadata</td>
</tr>
<tr>
<td style="text-align: center;"><code>annmap</code></td>
<td style="text-align: center;">1.38.0</td>
<td style="text-align: left;">Genome annotation and visualisation package pertaining to Affymetrix arrays and NGS analysis.</td>
</tr>
<tr>
<td style="text-align: center;"><code>annotate</code></td>
<td style="text-align: center;">1.74.0</td>
<td style="text-align: left;">Annotation for microarrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>AnnotationDbi</code></td>
<td style="text-align: center;">1.58.0</td>
<td style="text-align: left;">Manipulation of SQLite-based annotations in Bioconductor</td>
</tr>
<tr>
<td style="text-align: center;"><code>AnnotationFilter</code></td>
<td style="text-align: center;">1.20.0</td>
<td style="text-align: left;">Facilities for Filtering Bioconductor Annotation Resources</td>
</tr>
<tr>
<td style="text-align: center;"><code>AnnotationForge</code></td>
<td style="text-align: center;">1.38.0</td>
<td style="text-align: left;">Tools for building SQLite-based annotation data packages</td>
</tr>
<tr>
<td style="text-align: center;"><code>AnnotationHub</code></td>
<td style="text-align: center;">3.4.0</td>
<td style="text-align: left;">Client to access AnnotationHub resources</td>
</tr>
<tr>
<td style="text-align: center;"><code>ape</code></td>
<td style="text-align: center;">5.6-2</td>
<td style="text-align: left;">Analyses of Phylogenetics and Evolution</td>
</tr>
<tr>
<td style="text-align: center;"><code>arm</code></td>
<td style="text-align: center;">1.12-2</td>
<td style="text-align: left;">Data Analysis Using Regression and Multilevel/Hierarchical Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>aroma.affymetrix</code></td>
<td style="text-align: center;">3.2.0</td>
<td style="text-align: left;">Analysis of Large Affymetrix Microarray Data Sets</td>
</tr>
<tr>
<td style="text-align: center;"><code>aroma.apd</code></td>
<td style="text-align: center;">0.6.0</td>
<td style="text-align: left;">A Probe-Level Data File Format Used by 'aroma.affymetrix' [deprecated]</td>
</tr>
<tr>
<td style="text-align: center;"><code>aroma.core</code></td>
<td style="text-align: center;">3.2.2</td>
<td style="text-align: left;">Core Methods and Classes Used by 'aroma.*' Packages Part of the Aroma Framework</td>
</tr>
<tr>
<td style="text-align: center;"><code>aroma.light</code></td>
<td style="text-align: center;">3.26.0</td>
<td style="text-align: left;">Light-Weight Methods for Normalization and Visualization of Microarray Data using Only Basic R Data Types</td>
</tr>
<tr>
<td style="text-align: center;"><code>arrangements</code></td>
<td style="text-align: center;">1.1.9</td>
<td style="text-align: left;">Fast Generators and Iterators for Permutations, Combinations, Integer Partitions and Compositions</td>
</tr>
<tr>
<td style="text-align: center;"><code>arsenal</code></td>
<td style="text-align: center;">3.6.3</td>
<td style="text-align: left;">An Arsenal of 'R' Functions for Large-Scale Statistical Summaries</td>
</tr>
<tr>
<td style="text-align: center;"><code>askpass</code></td>
<td style="text-align: center;">1.1</td>
<td style="text-align: left;">Safe Password Entry for R, Git, and SSH</td>
</tr>
<tr>
<td style="text-align: center;"><code>assertthat</code></td>
<td style="text-align: center;">0.2.1</td>
<td style="text-align: left;">Easy Pre and Post Assertions</td>
</tr>
<tr>
<td style="text-align: center;"><code>backports</code></td>
<td style="text-align: center;">1.4.1</td>
<td style="text-align: left;">Reimplementations of Functions Introduced Since R-3.0.0</td>
</tr>
<tr>
<td style="text-align: center;"><code>bamsignals</code></td>
<td style="text-align: center;">1.28.0</td>
<td style="text-align: left;">Extract read count signals from bam files</td>
</tr>
<tr>
<td style="text-align: center;"><code>base</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">The R Base Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>base64</code></td>
<td style="text-align: center;">2.0</td>
<td style="text-align: left;">Base64 Encoder and Decoder</td>
</tr>
<tr>
<td style="text-align: center;"><code>base64enc</code></td>
<td style="text-align: center;">0.1-3</td>
<td style="text-align: left;">Tools for base64 encoding</td>
</tr>
<tr>
<td style="text-align: center;"><code>BaSTA</code></td>
<td style="text-align: center;">1.9.4</td>
<td style="text-align: left;">Age-Specific Survival Analysis from Incomplete Capture-Recapture/Recovery Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>BatchJobs</code></td>
<td style="text-align: center;">1.9</td>
<td style="text-align: left;">Batch Computing with R</td>
</tr>
<tr>
<td style="text-align: center;"><code>bayesplot</code></td>
<td style="text-align: center;">1.9.0</td>
<td style="text-align: left;">Plotting for Bayesian Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>BBmisc</code></td>
<td style="text-align: center;">1.12</td>
<td style="text-align: left;">Miscellaneous Helper Functions for B. Bischl</td>
</tr>
<tr>
<td style="text-align: center;"><code>beachmat</code></td>
<td style="text-align: center;">2.12.0</td>
<td style="text-align: left;">Compiling Bioconductor to Handle Each Matrix Type</td>
</tr>
<tr>
<td style="text-align: center;"><code>beadarray</code></td>
<td style="text-align: center;">2.46.0</td>
<td style="text-align: left;">Quality assessment and low-level analysis for Illumina BeadArray data</td>
</tr>
<tr>
<td style="text-align: center;"><code>beadarrayExampleData</code></td>
<td style="text-align: center;">1.34.0</td>
<td style="text-align: left;">Example data for the beadarray package</td>
</tr>
<tr>
<td style="text-align: center;"><code>BeadDataPackR</code></td>
<td style="text-align: center;">1.48.0</td>
<td style="text-align: left;">Compression of Illumina BeadArray data</td>
</tr>
<tr>
<td style="text-align: center;"><code>beanplot</code></td>
<td style="text-align: center;">1.3.1</td>
<td style="text-align: left;">Visualization via Beanplots (Like Boxplot/Stripchart/Violin Plot)</td>
</tr>
<tr>
<td style="text-align: center;"><code>benchmarkme</code></td>
<td style="text-align: center;">1.0.7</td>
<td style="text-align: left;">Crowd Sourced System Benchmarks</td>
</tr>
<tr>
<td style="text-align: center;"><code>benchmarkmeData</code></td>
<td style="text-align: center;">1.0.4</td>
<td style="text-align: left;">Data Set for the 'benchmarkme' Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>bezier</code></td>
<td style="text-align: center;">1.1.2</td>
<td style="text-align: left;">Toolkit for Bezier Curves and Splines</td>
</tr>
<tr>
<td style="text-align: center;"><code>BH</code></td>
<td style="text-align: center;">1.78.0-0</td>
<td style="text-align: left;">Boost C++ Header Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>BiasedUrn</code></td>
<td style="text-align: center;">1.07</td>
<td style="text-align: left;">Biased Urn Model Distributions</td>
</tr>
<tr>
<td style="text-align: center;"><code>binom</code></td>
<td style="text-align: center;">1.1-1.1</td>
<td style="text-align: left;">Binomial Confidence Intervals for Several Parameterizations</td>
</tr>
<tr>
<td style="text-align: center;"><code>bio3d</code></td>
<td style="text-align: center;">2.4-3</td>
<td style="text-align: left;">Biological Structure Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>Biobase</code></td>
<td style="text-align: center;">2.56.0</td>
<td style="text-align: left;">Biobase: Base functions for Bioconductor</td>
</tr>
<tr>
<td style="text-align: center;"><code>BiocFileCache</code></td>
<td style="text-align: center;">2.4.0</td>
<td style="text-align: left;">Manage Files Across Sessions</td>
</tr>
<tr>
<td style="text-align: center;"><code>BiocGenerics</code></td>
<td style="text-align: center;">0.42.0</td>
<td style="text-align: left;">S4 generic functions used in Bioconductor</td>
</tr>
<tr>
<td style="text-align: center;"><code>BiocIO</code></td>
<td style="text-align: center;">1.6.0</td>
<td style="text-align: left;">Standard Input and Output for Bioconductor Packages</td>
</tr>
<tr>
<td style="text-align: center;"><code>BiocManager</code></td>
<td style="text-align: center;">1.30.18</td>
<td style="text-align: left;">Access the Bioconductor Project Package Repository</td>
</tr>
<tr>
<td style="text-align: center;"><code>BiocParallel</code></td>
<td style="text-align: center;">1.30.2</td>
<td style="text-align: left;">Bioconductor facilities for parallel evaluation</td>
</tr>
<tr>
<td style="text-align: center;"><code>BiocVersion</code></td>
<td style="text-align: center;">3.15.2</td>
<td style="text-align: left;">Set the appropriate version of Bioconductor packages</td>
</tr>
<tr>
<td style="text-align: center;"><code>biomaRt</code></td>
<td style="text-align: center;">2.52.0</td>
<td style="text-align: left;">Interface to BioMart databases (i.e. Ensembl)</td>
</tr>
<tr>
<td style="text-align: center;"><code>Biostrings</code></td>
<td style="text-align: center;">2.64.0</td>
<td style="text-align: left;">Efficient manipulation of biological strings</td>
</tr>
<tr>
<td style="text-align: center;"><code>biovizBase</code></td>
<td style="text-align: center;">1.44.0</td>
<td style="text-align: left;">Basic graphic utilities for visualization of genomic data.</td>
</tr>
<tr>
<td style="text-align: center;"><code>bit</code></td>
<td style="text-align: center;">4.0.4</td>
<td style="text-align: left;">Classes and Methods for Fast Memory-Efficient Boolean Selections</td>
</tr>
<tr>
<td style="text-align: center;"><code>bit64</code></td>
<td style="text-align: center;">4.0.5</td>
<td style="text-align: left;">A S3 Class for Vectors of 64bit Integers</td>
</tr>
<tr>
<td style="text-align: center;"><code>bitops</code></td>
<td style="text-align: center;">1.0-7</td>
<td style="text-align: left;">Bitwise Operations</td>
</tr>
<tr>
<td style="text-align: center;"><code>blob</code></td>
<td style="text-align: center;">1.2.3</td>
<td style="text-align: left;">A Simple S3 Class for Representing Vectors of Binary Data ('BLOBS')</td>
</tr>
<tr>
<td style="text-align: center;"><code>blockmodeling</code></td>
<td style="text-align: center;">1.0.5</td>
<td style="text-align: left;">Generalized and Classical Blockmodeling of Valued Networks</td>
</tr>
<tr>
<td style="text-align: center;"><code>boot</code></td>
<td style="text-align: center;">1.3-28</td>
<td style="text-align: left;">Bootstrap Functions (Originally by Angelo Canty for S)</td>
</tr>
<tr>
<td style="text-align: center;"><code>BradleyTerry2</code></td>
<td style="text-align: center;">1.1-2</td>
<td style="text-align: left;">Bradley-Terry Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>brew</code></td>
<td style="text-align: center;">1.0-7</td>
<td style="text-align: left;">Templating Framework for Report Generation</td>
</tr>
<tr>
<td style="text-align: center;"><code>brglm</code></td>
<td style="text-align: center;">0.7.2</td>
<td style="text-align: left;">Bias Reduction in Binomial-Response Generalized Linear Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>bridgesampling</code></td>
<td style="text-align: center;">1.1-2</td>
<td style="text-align: left;">Bridge Sampling for Marginal Likelihoods and Bayes Factors</td>
</tr>
<tr>
<td style="text-align: center;"><code>brio</code></td>
<td style="text-align: center;">1.1.3</td>
<td style="text-align: left;">Basic R Input Output</td>
</tr>
<tr>
<td style="text-align: center;"><code>brms</code></td>
<td style="text-align: center;">2.18.0</td>
<td style="text-align: left;">Bayesian Regression Models using 'Stan'</td>
</tr>
<tr>
<td style="text-align: center;"><code>Brobdingnag</code></td>
<td style="text-align: center;">1.2-9</td>
<td style="text-align: left;">Very Large Numbers in R</td>
</tr>
<tr>
<td style="text-align: center;"><code>broom</code></td>
<td style="text-align: center;">0.8.0</td>
<td style="text-align: left;">Convert Statistical Objects into Tidy Tibbles</td>
</tr>
<tr>
<td style="text-align: center;"><code>BSgenome</code></td>
<td style="text-align: center;">1.64.0</td>
<td style="text-align: left;">Software infrastructure for efficient representation of full genomes and their SNPs</td>
</tr>
<tr>
<td style="text-align: center;"><code>BSgenome.Hsapiens.UCSC.hg19</code></td>
<td style="text-align: center;">1.4.3</td>
<td style="text-align: left;">Full genome sequences for Homo sapiens (UCSC version hg19, based on GRCh37.p13)</td>
</tr>
<tr>
<td style="text-align: center;"><code>bslib</code></td>
<td style="text-align: center;">0.3.1</td>
<td style="text-align: left;">Custom 'Bootstrap' 'Sass' Themes for 'shiny' and 'rmarkdown'</td>
</tr>
<tr>
<td style="text-align: center;"><code>bsseq</code></td>
<td style="text-align: center;">1.32.0</td>
<td style="text-align: left;">Analyze, manage and store bisulfite sequencing data</td>
</tr>
<tr>
<td style="text-align: center;"><code>bumphunter</code></td>
<td style="text-align: center;">1.38.0</td>
<td style="text-align: left;">Bump Hunter</td>
</tr>
<tr>
<td style="text-align: center;"><code>cachem</code></td>
<td style="text-align: center;">1.0.6</td>
<td style="text-align: left;">Cache R Objects with Automatic Pruning</td>
</tr>
<tr>
<td style="text-align: center;"><code>callr</code></td>
<td style="text-align: center;">3.7.0</td>
<td style="text-align: left;">Call R from R</td>
</tr>
<tr>
<td style="text-align: center;"><code>car</code></td>
<td style="text-align: center;">3.0-13</td>
<td style="text-align: left;">Companion to Applied Regression</td>
</tr>
<tr>
<td style="text-align: center;"><code>carData</code></td>
<td style="text-align: center;">3.0-5</td>
<td style="text-align: left;">Companion to Applied Regression Data Sets</td>
</tr>
<tr>
<td style="text-align: center;"><code>caret</code></td>
<td style="text-align: center;">6.0-92</td>
<td style="text-align: left;">Classification and Regression Training</td>
</tr>
<tr>
<td style="text-align: center;"><code>Category</code></td>
<td style="text-align: center;">2.62.0</td>
<td style="text-align: left;">Category Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>caTools</code></td>
<td style="text-align: center;">1.18.2</td>
<td style="text-align: left;">Tools: Moving Window Statistics, GIF, Base64, ROC AUC, etc</td>
</tr>
<tr>
<td style="text-align: center;"><code>CDM</code></td>
<td style="text-align: center;">8.1-12</td>
<td style="text-align: left;">Cognitive Diagnosis Modeling</td>
</tr>
<tr>
<td style="text-align: center;"><code>cellranger</code></td>
<td style="text-align: center;">1.1.0</td>
<td style="text-align: left;">Translate Spreadsheet Cell Ranges to Rows and Columns</td>
</tr>
<tr>
<td style="text-align: center;"><code>ChAMP</code></td>
<td style="text-align: center;">2.26.0</td>
<td style="text-align: left;">Chip Analysis Methylation Pipeline for Illumina HumanMethylation450 and EPIC</td>
</tr>
<tr>
<td style="text-align: center;"><code>ChAMPdata</code></td>
<td style="text-align: center;">2.28.0</td>
<td style="text-align: left;">Data Packages for ChAMP package</td>
</tr>
<tr>
<td style="text-align: center;"><code>checkmate</code></td>
<td style="text-align: center;">2.1.0</td>
<td style="text-align: left;">Fast and Versatile Argument Checks</td>
</tr>
<tr>
<td style="text-align: center;"><code>class</code></td>
<td style="text-align: center;">7.3-20</td>
<td style="text-align: left;">Functions for Classification</td>
</tr>
<tr>
<td style="text-align: center;"><code>classInt</code></td>
<td style="text-align: center;">0.4-3</td>
<td style="text-align: left;">Choose Univariate Class Intervals</td>
</tr>
<tr>
<td style="text-align: center;"><code>cli</code></td>
<td style="text-align: center;">3.6.0</td>
<td style="text-align: left;">Helpers for Developing Command Line Interfaces</td>
</tr>
<tr>
<td style="text-align: center;"><code>clipr</code></td>
<td style="text-align: center;">0.8.0</td>
<td style="text-align: left;">Read and Write from the System Clipboard</td>
</tr>
<tr>
<td style="text-align: center;"><code>clue</code></td>
<td style="text-align: center;">0.3-60</td>
<td style="text-align: left;">Cluster Ensembles</td>
</tr>
<tr>
<td style="text-align: center;"><code>cluster</code></td>
<td style="text-align: center;">2.1.3</td>
<td style="text-align: left;">"Finding Groups in Data": Cluster Analysis Extended Rousseeuw et al.</td>
</tr>
<tr>
<td style="text-align: center;"><code>cmprsk</code></td>
<td style="text-align: center;">2.2-11</td>
<td style="text-align: left;">Subdistribution Analysis of Competing Risks</td>
</tr>
<tr>
<td style="text-align: center;"><code>coda</code></td>
<td style="text-align: center;">0.19-4</td>
<td style="text-align: left;">Output Analysis and Diagnostics for MCMC</td>
</tr>
<tr>
<td style="text-align: center;"><code>codetools</code></td>
<td style="text-align: center;">0.2-18</td>
<td style="text-align: left;">Code Analysis Tools for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>colorRamps</code></td>
<td style="text-align: center;">2.3.1</td>
<td style="text-align: left;">Builds Color Tables</td>
</tr>
<tr>
<td style="text-align: center;"><code>colorspace</code></td>
<td style="text-align: center;">2.0-3</td>
<td style="text-align: left;">A Toolbox for Manipulating and Assessing Colors and Palettes</td>
</tr>
<tr>
<td style="text-align: center;"><code>colourpicker</code></td>
<td style="text-align: center;">1.1.1</td>
<td style="text-align: left;">A Colour Picker Tool for Shiny and for Selecting Colours in Plots</td>
</tr>
<tr>
<td style="text-align: center;"><code>combinat</code></td>
<td style="text-align: center;">0.0-8</td>
<td style="text-align: left;">combinatorics utilities</td>
</tr>
<tr>
<td style="text-align: center;"><code>commonmark</code></td>
<td style="text-align: center;">1.8.0</td>
<td style="text-align: left;">High Performance CommonMark and Github Markdown Rendering in R</td>
</tr>
<tr>
<td style="text-align: center;"><code>compiler</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">The R Compiler Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>copula</code></td>
<td style="text-align: center;">1.0-1</td>
<td style="text-align: left;">Multivariate Dependence with Copulas</td>
</tr>
<tr>
<td style="text-align: center;"><code>copynumber</code></td>
<td style="text-align: center;">1.36.0</td>
<td style="text-align: left;">Segmentation of single- and multi-track copy number data by penalized least squares regression.</td>
</tr>
<tr>
<td style="text-align: center;"><code>corpcor</code></td>
<td style="text-align: center;">1.6.10</td>
<td style="text-align: left;">Efficient Estimation of Covariance and (Partial) Correlation</td>
</tr>
<tr>
<td style="text-align: center;"><code>corrplot</code></td>
<td style="text-align: center;">0.92</td>
<td style="text-align: left;">Visualization of a Correlation Matrix</td>
</tr>
<tr>
<td style="text-align: center;"><code>cowplot</code></td>
<td style="text-align: center;">1.1.1</td>
<td style="text-align: left;">Streamlined Plot Theme and Plot Annotations for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>cpp11</code></td>
<td style="text-align: center;">0.4.2</td>
<td style="text-align: left;">A C++11 Interface for R's C Interface</td>
</tr>
<tr>
<td style="text-align: center;"><code>crayon</code></td>
<td style="text-align: center;">1.5.1</td>
<td style="text-align: left;">Colored Terminal Output</td>
</tr>
<tr>
<td style="text-align: center;"><code>credentials</code></td>
<td style="text-align: center;">1.3.2</td>
<td style="text-align: left;">Tools for Managing SSH and Git Credentials</td>
</tr>
<tr>
<td style="text-align: center;"><code>crosstalk</code></td>
<td style="text-align: center;">1.2.0</td>
<td style="text-align: left;">Inter-Widget Interactivity for HTML Widgets</td>
</tr>
<tr>
<td style="text-align: center;"><code>crul</code></td>
<td style="text-align: center;">1.2.0</td>
<td style="text-align: left;">HTTP Client</td>
</tr>
<tr>
<td style="text-align: center;"><code>curl</code></td>
<td style="text-align: center;">4.3.2</td>
<td style="text-align: left;">A Modern and Flexible Web Client for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>data.table</code></td>
<td style="text-align: center;">1.14.2</td>
<td style="text-align: left;">Extension of <code>data.frame</code></td>
</tr>
<tr>
<td style="text-align: center;"><code>datasets</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">The R Datasets Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>DBI</code></td>
<td style="text-align: center;">1.1.2</td>
<td style="text-align: left;">R Database Interface</td>
</tr>
<tr>
<td style="text-align: center;"><code>dbplyr</code></td>
<td style="text-align: center;">2.1.1</td>
<td style="text-align: left;">A 'dplyr' Back End for Databases</td>
</tr>
<tr>
<td style="text-align: center;"><code>DelayedArray</code></td>
<td style="text-align: center;">0.22.0</td>
<td style="text-align: left;">A unified framework for working transparently with on-disk and in-memory array-like datasets</td>
</tr>
<tr>
<td style="text-align: center;"><code>DelayedMatrixStats</code></td>
<td style="text-align: center;">1.18.0</td>
<td style="text-align: left;">Functions that Apply to Rows and Columns of 'DelayedMatrix' Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>deldir</code></td>
<td style="text-align: center;">1.0-6</td>
<td style="text-align: left;">Delaunay Triangulation and Dirichlet (Voronoi) Tessellation</td>
</tr>
<tr>
<td style="text-align: center;"><code>dendextend</code></td>
<td style="text-align: center;">1.15.2</td>
<td style="text-align: left;">Extending 'dendrogram' Functionality in R</td>
</tr>
<tr>
<td style="text-align: center;"><code>DEoptimR</code></td>
<td style="text-align: center;">1.0-11</td>
<td style="text-align: left;">Differential Evolution Optimization in Pure R</td>
</tr>
<tr>
<td style="text-align: center;"><code>Deriv</code></td>
<td style="text-align: center;">4.1.3</td>
<td style="text-align: left;">Symbolic Differentiation</td>
</tr>
<tr>
<td style="text-align: center;"><code>desc</code></td>
<td style="text-align: center;">1.4.1</td>
<td style="text-align: left;">Manipulate DESCRIPTION Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>DescTools</code></td>
<td style="text-align: center;">0.99.47</td>
<td style="text-align: left;">Tools for Descriptive Statistics</td>
</tr>
<tr>
<td style="text-align: center;"><code>DESeq2</code></td>
<td style="text-align: center;">1.36.0</td>
<td style="text-align: left;">Differential gene expression analysis based on the negative binomial distribution</td>
</tr>
<tr>
<td style="text-align: center;"><code>devtools</code></td>
<td style="text-align: center;">2.4.3</td>
<td style="text-align: left;">Tools to Make Developing R Packages Easier</td>
</tr>
<tr>
<td style="text-align: center;"><code>DEXSeq</code></td>
<td style="text-align: center;">1.42.0</td>
<td style="text-align: left;">Inference of differential exon usage in RNA-Seq</td>
</tr>
<tr>
<td style="text-align: center;"><code>dichromat</code></td>
<td style="text-align: center;">2.0-0.1</td>
<td style="text-align: left;">Color Schemes for Dichromats</td>
</tr>
<tr>
<td style="text-align: center;"><code>diffobj</code></td>
<td style="text-align: center;">0.3.5</td>
<td style="text-align: left;">Diffs for R Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>digest</code></td>
<td style="text-align: center;">0.6.29</td>
<td style="text-align: left;">Create Compact Hash Digests of R Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>diptest</code></td>
<td style="text-align: center;">0.76-0</td>
<td style="text-align: left;">Hartigan's Dip Test Statistic for Unimodality - Corrected</td>
</tr>
<tr>
<td style="text-align: center;"><code>distr</code></td>
<td style="text-align: center;">2.8.0</td>
<td style="text-align: left;">Object Oriented Implementation of Distributions</td>
</tr>
<tr>
<td style="text-align: center;"><code>distrEx</code></td>
<td style="text-align: center;">2.8.0</td>
<td style="text-align: left;">Extensions of Package 'distr'</td>
</tr>
<tr>
<td style="text-align: center;"><code>distributional</code></td>
<td style="text-align: center;">0.3.0</td>
<td style="text-align: left;">Vectorised Probability Distributions</td>
</tr>
<tr>
<td style="text-align: center;"><code>DMRcate</code></td>
<td style="text-align: center;">2.10.0</td>
<td style="text-align: left;">Methylation array and sequencing spatial analysis methods</td>
</tr>
<tr>
<td style="text-align: center;"><code>DNAcopy</code></td>
<td style="text-align: center;">1.70.0</td>
<td style="text-align: left;">DNA copy number data analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>docopt</code></td>
<td style="text-align: center;">0.7.1</td>
<td style="text-align: left;">Command-Line Interface Specification Language</td>
</tr>
<tr>
<td style="text-align: center;"><code>doMC</code></td>
<td style="text-align: center;">1.3.8</td>
<td style="text-align: left;">Foreach Parallel Adaptor for 'parallel'</td>
</tr>
<tr>
<td style="text-align: center;"><code>doMPI</code></td>
<td style="text-align: center;">0.2.2</td>
<td style="text-align: left;">Foreach Parallel Adaptor for the Rmpi Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>doParallel</code></td>
<td style="text-align: center;">1.0.17</td>
<td style="text-align: left;">Foreach Parallel Adaptor for the 'parallel' Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>doRNG</code></td>
<td style="text-align: center;">1.8.2</td>
<td style="text-align: left;">Generic Reproducible Parallel Backend for 'foreach' Loops</td>
</tr>
<tr>
<td style="text-align: center;"><code>dotCall64</code></td>
<td style="text-align: center;">1.0-1</td>
<td style="text-align: left;">Enhanced Foreign Function Interface Supporting Long Vectors</td>
</tr>
<tr>
<td style="text-align: center;"><code>dparser</code></td>
<td style="text-align: center;">1.3.1-9</td>
<td style="text-align: left;">Port of 'Dparser' Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>dplyr</code></td>
<td style="text-align: center;">1.0.9</td>
<td style="text-align: left;">A Grammar of Data Manipulation</td>
</tr>
<tr>
<td style="text-align: center;"><code>DSS</code></td>
<td style="text-align: center;">2.44.0</td>
<td style="text-align: left;">Dispersion shrinkage for sequencing data</td>
</tr>
<tr>
<td style="text-align: center;"><code>DT</code></td>
<td style="text-align: center;">0.23</td>
<td style="text-align: left;">A Wrapper of the JavaScript Library 'DataTables'</td>
</tr>
<tr>
<td style="text-align: center;"><code>dtplyr</code></td>
<td style="text-align: center;">1.2.1</td>
<td style="text-align: left;">Data Table Back-End for 'dplyr'</td>
</tr>
<tr>
<td style="text-align: center;"><code>dygraphs</code></td>
<td style="text-align: center;">1.1.1.6</td>
<td style="text-align: left;">Interface to 'Dygraphs' Interactive Time Series Charting Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>dynamicTreeCut</code></td>
<td style="text-align: center;">1.63-1</td>
<td style="text-align: left;">Methods for Detection of Clusters in Hierarchical Clustering Dendrograms</td>
</tr>
<tr>
<td style="text-align: center;"><code>DynDoc</code></td>
<td style="text-align: center;">1.74.0</td>
<td style="text-align: left;">Dynamic document tools</td>
</tr>
<tr>
<td style="text-align: center;"><code>e1071</code></td>
<td style="text-align: center;">1.7-9</td>
<td style="text-align: left;">Misc Functions of the Department of Statistics, Probability Theory Group (Formerly: E1071), TU Wien</td>
</tr>
<tr>
<td style="text-align: center;"><code>easyRNASeq</code></td>
<td style="text-align: center;">2.32.0</td>
<td style="text-align: left;">Count summarization and normalization for RNA-Seq data</td>
</tr>
<tr>
<td style="text-align: center;"><code>EBSeq</code></td>
<td style="text-align: center;">1.36.0</td>
<td style="text-align: left;">An R package for gene and isoform differential expression analysis of RNA-seq data</td>
</tr>
<tr>
<td style="text-align: center;"><code>edgeR</code></td>
<td style="text-align: center;">3.38.1</td>
<td style="text-align: left;">Empirical Analysis of Digital Gene Expression Data in R</td>
</tr>
<tr>
<td style="text-align: center;"><code>effects</code></td>
<td style="text-align: center;">4.2-1</td>
<td style="text-align: left;">Effect Displays for Linear, Generalized Linear, and Other Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>ellipse</code></td>
<td style="text-align: center;">0.4.2</td>
<td style="text-align: left;">Functions for Drawing Ellipses and Ellipse-Like Confidence Regions</td>
</tr>
<tr>
<td style="text-align: center;"><code>ellipsis</code></td>
<td style="text-align: center;">0.3.2</td>
<td style="text-align: left;">Tools for Working with ...</td>
</tr>
<tr>
<td style="text-align: center;"><code>ensembldb</code></td>
<td style="text-align: center;">2.20.1</td>
<td style="text-align: left;">Utilities to create and use Ensembl-based annotation databases</td>
</tr>
<tr>
<td style="text-align: center;"><code>Epi</code></td>
<td style="text-align: center;">2.46</td>
<td style="text-align: left;">Statistical Analysis in Epidemiology</td>
</tr>
<tr>
<td style="text-align: center;"><code>erer</code></td>
<td style="text-align: center;">3.1</td>
<td style="text-align: left;">Empirical Research in Economics with R</td>
</tr>
<tr>
<td style="text-align: center;"><code>estimability</code></td>
<td style="text-align: center;">1.3</td>
<td style="text-align: left;">Tools for Assessing Estimability of Linear Predictions</td>
</tr>
<tr>
<td style="text-align: center;"><code>etm</code></td>
<td style="text-align: center;">1.1.1</td>
<td style="text-align: left;">Empirical Transition Matrix</td>
</tr>
<tr>
<td style="text-align: center;"><code>evaluate</code></td>
<td style="text-align: center;">0.15</td>
<td style="text-align: left;">Parsing and Evaluation Tools that Provide More Details than the Default</td>
</tr>
<tr>
<td style="text-align: center;"><code>evd</code></td>
<td style="text-align: center;">2.3-6</td>
<td style="text-align: left;">Functions for Extreme Value Distributions</td>
</tr>
<tr>
<td style="text-align: center;"><code>Exact</code></td>
<td style="text-align: center;">3.2</td>
<td style="text-align: left;">Unconditional Exact Test</td>
</tr>
<tr>
<td style="text-align: center;"><code>ExperimentHub</code></td>
<td style="text-align: center;">2.4.0</td>
<td style="text-align: left;">Client to access ExperimentHub resources</td>
</tr>
<tr>
<td style="text-align: center;"><code>expm</code></td>
<td style="text-align: center;">0.999-6</td>
<td style="text-align: left;">Matrix Exponential, Log, 'etc'</td>
</tr>
<tr>
<td style="text-align: center;"><code>FactoMineR</code></td>
<td style="text-align: center;">2.4</td>
<td style="text-align: left;">Multivariate Exploratory Data Analysis and Data Mining</td>
</tr>
<tr>
<td style="text-align: center;"><code>fail</code></td>
<td style="text-align: center;">1.3</td>
<td style="text-align: left;">File Abstraction Interface Layer (FAIL)</td>
</tr>
<tr>
<td style="text-align: center;"><code>fansi</code></td>
<td style="text-align: center;">1.0.3</td>
<td style="text-align: left;">ANSI Control Sequence Aware String Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>farver</code></td>
<td style="text-align: center;">2.1.0</td>
<td style="text-align: left;">High Performance Colour Space Manipulation</td>
</tr>
<tr>
<td style="text-align: center;"><code>fastcluster</code></td>
<td style="text-align: center;">1.2.3</td>
<td style="text-align: left;">Fast Hierarchical Clustering Routines for R and 'Python'</td>
</tr>
<tr>
<td style="text-align: center;"><code>fastICA</code></td>
<td style="text-align: center;">1.2-3</td>
<td style="text-align: left;">FastICA Algorithms to Perform ICA and Projection Pursuit</td>
</tr>
<tr>
<td style="text-align: center;"><code>fastmap</code></td>
<td style="text-align: center;">1.1.0</td>
<td style="text-align: left;">Fast Data Structures</td>
</tr>
<tr>
<td style="text-align: center;"><code>fastmatch</code></td>
<td style="text-align: center;">1.1-3</td>
<td style="text-align: left;">Fast 'match()' Function</td>
</tr>
<tr>
<td style="text-align: center;"><code>FDb.InfiniumMethylation.hg19</code></td>
<td style="text-align: center;">2.2.0</td>
<td style="text-align: left;">Annotation package for Illumina Infinium DNA methylation probes</td>
</tr>
<tr>
<td style="text-align: center;"><code>fdrtool</code></td>
<td style="text-align: center;">1.2.17</td>
<td style="text-align: left;">Estimation of (Local) False Discovery Rates and Higher Criticism</td>
</tr>
<tr>
<td style="text-align: center;"><code>fields</code></td>
<td style="text-align: center;">13.3</td>
<td style="text-align: left;">Tools for Spatial Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>filelock</code></td>
<td style="text-align: center;">1.0.2</td>
<td style="text-align: left;">Portable File Locking</td>
</tr>
<tr>
<td style="text-align: center;"><code>flashClust</code></td>
<td style="text-align: center;">1.01-2</td>
<td style="text-align: left;">Implementation of optimal hierarchical clustering</td>
</tr>
<tr>
<td style="text-align: center;"><code>flexmix</code></td>
<td style="text-align: center;">2.3-17</td>
<td style="text-align: left;">Flexible Mixture Modeling</td>
</tr>
<tr>
<td style="text-align: center;"><code>fontawesome</code></td>
<td style="text-align: center;">0.2.2</td>
<td style="text-align: left;">Easily Work with 'Font Awesome' Icons</td>
</tr>
<tr>
<td style="text-align: center;"><code>forcats</code></td>
<td style="text-align: center;">0.5.1</td>
<td style="text-align: left;">Tools for Working with Categorical Variables (Factors)</td>
</tr>
<tr>
<td style="text-align: center;"><code>foreach</code></td>
<td style="text-align: center;">1.5.2</td>
<td style="text-align: left;">Provides Foreach Looping Construct</td>
</tr>
<tr>
<td style="text-align: center;"><code>foreign</code></td>
<td style="text-align: center;">0.8-82</td>
<td style="text-align: left;">Read Data Stored by 'Minitab', 'S', 'SAS', 'SPSS', 'Stata', 'Systat', 'Weka', 'dBase', ...</td>
</tr>
<tr>
<td style="text-align: center;"><code>formatR</code></td>
<td style="text-align: center;">1.12</td>
<td style="text-align: left;">Format R Code Automatically</td>
</tr>
<tr>
<td style="text-align: center;"><code>Formula</code></td>
<td style="text-align: center;">1.2-4</td>
<td style="text-align: left;">Extended Model Formulas</td>
</tr>
<tr>
<td style="text-align: center;"><code>fpc</code></td>
<td style="text-align: center;">2.2-9</td>
<td style="text-align: left;">Flexible Procedures for Clustering</td>
</tr>
<tr>
<td style="text-align: center;"><code>fs</code></td>
<td style="text-align: center;">1.5.2</td>
<td style="text-align: left;">Cross-Platform File System Operations Based on 'libuv'</td>
</tr>
<tr>
<td style="text-align: center;"><code>futile.logger</code></td>
<td style="text-align: center;">1.4.3</td>
<td style="text-align: left;">A Logging Utility for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>futile.options</code></td>
<td style="text-align: center;">1.0.1</td>
<td style="text-align: left;">Futile Options Management</td>
</tr>
<tr>
<td style="text-align: center;"><code>future</code></td>
<td style="text-align: center;">1.25.0</td>
<td style="text-align: left;">Unified Parallel and Distributed Processing in R for Everyone</td>
</tr>
<tr>
<td style="text-align: center;"><code>future.apply</code></td>
<td style="text-align: center;">1.9.0</td>
<td style="text-align: left;">Apply Function to Elements in Parallel using Futures</td>
</tr>
<tr>
<td style="text-align: center;"><code>gam</code></td>
<td style="text-align: center;">1.20.1</td>
<td style="text-align: left;">Generalized Additive Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>gamlss</code></td>
<td style="text-align: center;">5.4-3</td>
<td style="text-align: left;">Generalised Additive Models for Location Scale and Shape</td>
</tr>
<tr>
<td style="text-align: center;"><code>gamlss.data</code></td>
<td style="text-align: center;">6.0-2</td>
<td style="text-align: left;">Data for Generalised Additive Models for Location Scale and Shape</td>
</tr>
<tr>
<td style="text-align: center;"><code>gamlss.dist</code></td>
<td style="text-align: center;">6.0-3</td>
<td style="text-align: left;">Distributions for Generalized Additive Models for Location Scale and Shape</td>
</tr>
<tr>
<td style="text-align: center;"><code>gamlss.mx</code></td>
<td style="text-align: center;">6.0-0</td>
<td style="text-align: left;">Fitting Mixture Distributions with GAMLSS</td>
</tr>
<tr>
<td style="text-align: center;"><code>gamlss.nl</code></td>
<td style="text-align: center;">4.1-0</td>
<td style="text-align: left;">Fitting non linear parametric GAMLSS models</td>
</tr>
<tr>
<td style="text-align: center;"><code>gargle</code></td>
<td style="text-align: center;">1.2.0</td>
<td style="text-align: left;">Utilities for Working with Google APIs</td>
</tr>
<tr>
<td style="text-align: center;"><code>gcrma</code></td>
<td style="text-align: center;">2.68.0</td>
<td style="text-align: left;">Background Adjustment Using Sequence Information</td>
</tr>
<tr>
<td style="text-align: center;"><code>gdata</code></td>
<td style="text-align: center;">2.18.0.1</td>
<td style="text-align: left;">Various R Programming Tools for Data Manipulation</td>
</tr>
<tr>
<td style="text-align: center;"><code>genefilter</code></td>
<td style="text-align: center;">1.78.0</td>
<td style="text-align: left;">genefilter: methods for filtering genes from high-throughput experiments</td>
</tr>
<tr>
<td style="text-align: center;"><code>geneLenDataBase</code></td>
<td style="text-align: center;">1.32.0</td>
<td style="text-align: left;">Lengths of mRNA transcripts for a number of genomes</td>
</tr>
<tr>
<td style="text-align: center;"><code>GeneNet</code></td>
<td style="text-align: center;">1.2.16</td>
<td style="text-align: left;">Modeling and Inferring Gene Networks</td>
</tr>
<tr>
<td style="text-align: center;"><code>geneplotter</code></td>
<td style="text-align: center;">1.74.0</td>
<td style="text-align: left;">Graphics related functions for Bioconductor</td>
</tr>
<tr>
<td style="text-align: center;"><code>generics</code></td>
<td style="text-align: center;">0.1.2</td>
<td style="text-align: left;">Common S3 Generics not Provided by Base R Methods Related to Model Fitting</td>
</tr>
<tr>
<td style="text-align: center;"><code>genetics</code></td>
<td style="text-align: center;">1.3.8.1.3</td>
<td style="text-align: left;">Population Genetics</td>
</tr>
<tr>
<td style="text-align: center;"><code>GenomeInfoDb</code></td>
<td style="text-align: center;">1.32.2</td>
<td style="text-align: left;">Utilities for manipulating chromosome names, including modifying them to follow a particular naming style</td>
</tr>
<tr>
<td style="text-align: center;"><code>GenomeInfoDbData</code></td>
<td style="text-align: center;">1.2.8</td>
<td style="text-align: left;">Species and taxonomy ID look up tables used by GenomeInfoDb</td>
</tr>
<tr>
<td style="text-align: center;"><code>genomeIntervals</code></td>
<td style="text-align: center;">1.52.0</td>
<td style="text-align: left;">Operations on genomic intervals</td>
</tr>
<tr>
<td style="text-align: center;"><code>GenomicAlignments</code></td>
<td style="text-align: center;">1.32.0</td>
<td style="text-align: left;">Representation and manipulation of short genomic alignments</td>
</tr>
<tr>
<td style="text-align: center;"><code>GenomicFeatures</code></td>
<td style="text-align: center;">1.48.1</td>
<td style="text-align: left;">Conveniently import and query gene models</td>
</tr>
<tr>
<td style="text-align: center;"><code>GenomicRanges</code></td>
<td style="text-align: center;">1.48.0</td>
<td style="text-align: left;">Representation and manipulation of genomic intervals</td>
</tr>
<tr>
<td style="text-align: center;"><code>geojson</code></td>
<td style="text-align: center;">0.3.4</td>
<td style="text-align: left;">Classes for 'GeoJSON'</td>
</tr>
<tr>
<td style="text-align: center;"><code>geojsonio</code></td>
<td style="text-align: center;">0.9.4</td>
<td style="text-align: left;">Convert Data from and to 'GeoJSON' or 'TopoJSON'</td>
</tr>
<tr>
<td style="text-align: center;"><code>geojsonsf</code></td>
<td style="text-align: center;">2.0.2</td>
<td style="text-align: left;">GeoJSON to Simple Feature Converter</td>
</tr>
<tr>
<td style="text-align: center;"><code>geometries</code></td>
<td style="text-align: center;">0.2.0</td>
<td style="text-align: left;">Convert Between R Objects and Geometric Structures</td>
</tr>
<tr>
<td style="text-align: center;"><code>GEOquery</code></td>
<td style="text-align: center;">2.64.2</td>
<td style="text-align: left;">Get data from NCBI Gene Expression Omnibus (GEO)</td>
</tr>
<tr>
<td style="text-align: center;"><code>gert</code></td>
<td style="text-align: center;">1.6.0</td>
<td style="text-align: left;">Simple Git Client for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>getopt</code></td>
<td style="text-align: center;">1.20.3</td>
<td style="text-align: left;">C-Like 'getopt' Behavior</td>
</tr>
<tr>
<td style="text-align: center;"><code>GGally</code></td>
<td style="text-align: center;">2.1.2</td>
<td style="text-align: left;">Extension to 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>ggforce</code></td>
<td style="text-align: center;">0.3.3</td>
<td style="text-align: left;">Accelerating 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>ggplot2</code></td>
<td style="text-align: center;">3.3.6</td>
<td style="text-align: left;">Create Elegant Data Visualisations Using the Grammar of Graphics</td>
</tr>
<tr>
<td style="text-align: center;"><code>ggrepel</code></td>
<td style="text-align: center;">0.9.1</td>
<td style="text-align: left;">Automatically Position Non-Overlapping Text Labels with 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>ggridges</code></td>
<td style="text-align: center;">0.5.3</td>
<td style="text-align: left;">Ridgeline Plots in 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>ggtext</code></td>
<td style="text-align: center;">0.1.2</td>
<td style="text-align: left;">Improved Text Rendering Support for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>gh</code></td>
<td style="text-align: center;">1.3.0</td>
<td style="text-align: left;">'GitHub' 'API'</td>
</tr>
<tr>
<td style="text-align: center;"><code>gitcreds</code></td>
<td style="text-align: center;">0.1.1</td>
<td style="text-align: left;">Query 'git' Credentials from 'R'</td>
</tr>
<tr>
<td style="text-align: center;"><code>GJRM</code></td>
<td style="text-align: center;">0.2-6</td>
<td style="text-align: left;">Generalised Joint Regression Modelling</td>
</tr>
<tr>
<td style="text-align: center;"><code>gld</code></td>
<td style="text-align: center;">2.6.6</td>
<td style="text-align: left;">Estimation and Use of the Generalised (Tukey) Lambda Distribution</td>
</tr>
<tr>
<td style="text-align: center;"><code>glmnet</code></td>
<td style="text-align: center;">4.1-4</td>
<td style="text-align: left;">Lasso and Elastic-Net Regularized Generalized Linear Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>globals</code></td>
<td style="text-align: center;">0.15.0</td>
<td style="text-align: left;">Identify Global Objects in R Expressions</td>
</tr>
<tr>
<td style="text-align: center;"><code>globaltest</code></td>
<td style="text-align: center;">5.50.0</td>
<td style="text-align: left;">Testing Groups of Covariates/Features for Association with a Response Variable, with Applications to Gene Set Testing</td>
</tr>
<tr>
<td style="text-align: center;"><code>glue</code></td>
<td style="text-align: center;">1.6.2</td>
<td style="text-align: left;">Interpreted String Literals</td>
</tr>
<tr>
<td style="text-align: center;"><code>gmp</code></td>
<td style="text-align: center;">0.6-5</td>
<td style="text-align: left;">Multiple Precision Arithmetic</td>
</tr>
<tr>
<td style="text-align: center;"><code>GO.db</code></td>
<td style="text-align: center;">3.15.0</td>
<td style="text-align: left;">A set of annotation maps describing the entire Gene Ontology</td>
</tr>
<tr>
<td style="text-align: center;"><code>goftest</code></td>
<td style="text-align: center;">1.2-3</td>
<td style="text-align: left;">Classical Goodness-of-Fit Tests for Univariate Distributions</td>
</tr>
<tr>
<td style="text-align: center;"><code>googledrive</code></td>
<td style="text-align: center;">2.0.0</td>
<td style="text-align: left;">An Interface to Google Drive</td>
</tr>
<tr>
<td style="text-align: center;"><code>googlesheets4</code></td>
<td style="text-align: center;">1.0.0</td>
<td style="text-align: left;">Access Google Sheets using the Sheets API V4</td>
</tr>
<tr>
<td style="text-align: center;"><code>goseq</code></td>
<td style="text-align: center;">1.48.0</td>
<td style="text-align: left;">Gene Ontology analyser for RNA-seq and other length biased data</td>
</tr>
<tr>
<td style="text-align: center;"><code>GOstats</code></td>
<td style="text-align: center;">2.62.0</td>
<td style="text-align: left;">Tools for manipulating GO and microarrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>gower</code></td>
<td style="text-align: center;">1.0.0</td>
<td style="text-align: left;">Gower's Distance</td>
</tr>
<tr>
<td style="text-align: center;"><code>gplots</code></td>
<td style="text-align: center;">3.1.3</td>
<td style="text-align: left;">Various R Programming Tools for Plotting Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>graph</code></td>
<td style="text-align: center;">1.74.0</td>
<td style="text-align: left;">graph: A package to handle graph data structures</td>
</tr>
<tr>
<td style="text-align: center;"><code>graphics</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">The R Graphics Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>grDevices</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">The R Graphics Devices and Support for Colours and Fonts</td>
</tr>
<tr>
<td style="text-align: center;"><code>grid</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">The Grid Graphics Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>gridExtra</code></td>
<td style="text-align: center;">2.3</td>
<td style="text-align: left;">Miscellaneous Functions for "Grid" Graphics</td>
</tr>
<tr>
<td style="text-align: center;"><code>gridtext</code></td>
<td style="text-align: center;">0.1.5</td>
<td style="text-align: left;">Improved Text Rendering Support for 'Grid' Graphics</td>
</tr>
<tr>
<td style="text-align: center;"><code>GSEABase</code></td>
<td style="text-align: center;">1.58.0</td>
<td style="text-align: left;">Gene set enrichment data structures and methods</td>
</tr>
<tr>
<td style="text-align: center;"><code>gsl</code></td>
<td style="text-align: center;">2.1-7.1</td>
<td style="text-align: left;">Wrapper for the Gnu Scientific Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>gsmoothr</code></td>
<td style="text-align: center;">0.1.7</td>
<td style="text-align: left;">Smoothing tools</td>
</tr>
<tr>
<td style="text-align: center;"><code>gtable</code></td>
<td style="text-align: center;">0.3.0</td>
<td style="text-align: left;">Arrange 'Grobs' in Tables</td>
</tr>
<tr>
<td style="text-align: center;"><code>gtools</code></td>
<td style="text-align: center;">3.9.2.1</td>
<td style="text-align: left;">Various R Programming Tools</td>
</tr>
<tr>
<td style="text-align: center;"><code>Gviz</code></td>
<td style="text-align: center;">1.40.1</td>
<td style="text-align: left;">Plotting data and annotation information along genomic coordinates</td>
</tr>
<tr>
<td style="text-align: center;"><code>HAC</code></td>
<td style="text-align: center;">1.1-0</td>
<td style="text-align: left;">Estimation, Simulation and Visualization of Hierarchical Archimedean Copulae (HAC)</td>
</tr>
<tr>
<td style="text-align: center;"><code>haplo.stats</code></td>
<td style="text-align: center;">1.8.7</td>
<td style="text-align: left;">Statistical Analysis of Haplotypes with Traits and Covariates when Linkage Phase is Ambiguous</td>
</tr>
<tr>
<td style="text-align: center;"><code>hardhat</code></td>
<td style="text-align: center;">0.2.0</td>
<td style="text-align: left;">Construct Modeling Packages</td>
</tr>
<tr>
<td style="text-align: center;"><code>haven</code></td>
<td style="text-align: center;">2.5.0</td>
<td style="text-align: left;">Import and Export 'SPSS', 'Stata' and 'SAS' Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>HDF5Array</code></td>
<td style="text-align: center;">1.24.0</td>
<td style="text-align: left;">HDF5 backend for DelayedArray objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>hexbin</code></td>
<td style="text-align: center;">1.28.2</td>
<td style="text-align: left;">Hexagonal Binning Routines</td>
</tr>
<tr>
<td style="text-align: center;"><code>hgu95av2.db</code></td>
<td style="text-align: center;">3.13.0</td>
<td style="text-align: left;">Affymetrix Affymetrix HG_U95Av2 Array annotation data (chip hgu95av2)</td>
</tr>
<tr>
<td style="text-align: center;"><code>HI</code></td>
<td style="text-align: center;">0.5</td>
<td style="text-align: left;">Simulation from Distributions Supported by Nested Hyperplanes</td>
</tr>
<tr>
<td style="text-align: center;"><code>highr</code></td>
<td style="text-align: center;">0.9</td>
<td style="text-align: left;">Syntax Highlighting for R Source Code</td>
</tr>
<tr>
<td style="text-align: center;"><code>HistData</code></td>
<td style="text-align: center;">0.8-7</td>
<td style="text-align: left;">Data Sets from the History of Statistics and Data Visualization</td>
</tr>
<tr>
<td style="text-align: center;"><code>Hmisc</code></td>
<td style="text-align: center;">4.7-0</td>
<td style="text-align: left;">Harrell Miscellaneous</td>
</tr>
<tr>
<td style="text-align: center;"><code>hms</code></td>
<td style="text-align: center;">1.1.1</td>
<td style="text-align: left;">Pretty Time of Day</td>
</tr>
<tr>
<td style="text-align: center;"><code>htmlTable</code></td>
<td style="text-align: center;">2.4.0</td>
<td style="text-align: left;">Advanced Tables for Markdown/HTML</td>
</tr>
<tr>
<td style="text-align: center;"><code>htmltools</code></td>
<td style="text-align: center;">0.5.2</td>
<td style="text-align: left;">Tools for HTML</td>
</tr>
<tr>
<td style="text-align: center;"><code>htmlwidgets</code></td>
<td style="text-align: center;">1.5.4</td>
<td style="text-align: left;">HTML Widgets for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>httpcode</code></td>
<td style="text-align: center;">0.3.0</td>
<td style="text-align: left;">'HTTP' Status Code Helper</td>
</tr>
<tr>
<td style="text-align: center;"><code>httpuv</code></td>
<td style="text-align: center;">1.6.5</td>
<td style="text-align: left;">HTTP and WebSocket Server Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>httr</code></td>
<td style="text-align: center;">1.4.3</td>
<td style="text-align: left;">Tools for Working with URLs and HTTP</td>
</tr>
<tr>
<td style="text-align: center;"><code>hugene10stprobeset.db</code></td>
<td style="text-align: center;">8.8.0</td>
<td style="text-align: left;">Affymetrix hugene10 annotation data (chip hugene10stprobeset)</td>
</tr>
<tr>
<td style="text-align: center;"><code>hugene10sttranscriptcluster.db</code></td>
<td style="text-align: center;">8.8.0</td>
<td style="text-align: left;">Affymetrix hugene10 annotation data (chip hugene10sttranscriptcluster)</td>
</tr>
<tr>
<td style="text-align: center;"><code>hwriter</code></td>
<td style="text-align: center;">1.3.2.1</td>
<td style="text-align: left;">HTML Writer - Outputs R Objects in HTML Format</td>
</tr>
<tr>
<td style="text-align: center;"><code>ids</code></td>
<td style="text-align: center;">1.0.1</td>
<td style="text-align: left;">Generate Random Identifiers</td>
</tr>
<tr>
<td style="text-align: center;"><code>igraph</code></td>
<td style="text-align: center;">1.3.1</td>
<td style="text-align: left;">Network Analysis and Visualization</td>
</tr>
<tr>
<td style="text-align: center;"><code>Illumina450ProbeVariants.db</code></td>
<td style="text-align: center;">1.32.0</td>
<td style="text-align: left;">Annotation Package combining variant data from 1000 Genomes Project for Illumina HumanMethylation450 Bead Chip probes</td>
</tr>
<tr>
<td style="text-align: center;"><code>IlluminaHumanMethylation450kanno.ilmn12.hg19</code></td>
<td style="text-align: center;">0.6.1</td>
<td style="text-align: left;">Annotation for Illumina's 450k methylation arrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>IlluminaHumanMethylation450kmanifest</code></td>
<td style="text-align: center;">0.4.0</td>
<td style="text-align: left;">Annotation for Illumina's 450k methylation arrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>IlluminaHumanMethylationEPICanno.ilm10b4.hg19</code></td>
<td style="text-align: center;">0.6.0</td>
<td style="text-align: left;">Annotation for Illumina's EPIC methylation arrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>IlluminaHumanMethylationEPICmanifest</code></td>
<td style="text-align: center;">0.3.0</td>
<td style="text-align: left;">Manifest for Illumina's EPIC methylation arrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>illuminaHumanv4.db</code></td>
<td style="text-align: center;">1.26.0</td>
<td style="text-align: left;">Illumina HumanHT12v4 annotation data (chip illuminaHumanv4)</td>
</tr>
<tr>
<td style="text-align: center;"><code>illuminaio</code></td>
<td style="text-align: center;">0.38.0</td>
<td style="text-align: left;">Parsing Illumina Microarray Output Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>impute</code></td>
<td style="text-align: center;">1.70.0</td>
<td style="text-align: left;">impute: Imputation for microarray data</td>
</tr>
<tr>
<td style="text-align: center;"><code>ini</code></td>
<td style="text-align: center;">0.3.1</td>
<td style="text-align: left;">Read and Write '.ini' Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>INLA</code></td>
<td style="text-align: center;">22.05.18-2</td>
<td style="text-align: left;">Full Bayesian Analysis of Latent Gaussian Models using Integrated Nested Laplace Approximations</td>
</tr>
<tr>
<td style="text-align: center;"><code>inline</code></td>
<td style="text-align: center;">0.3.19</td>
<td style="text-align: left;">Functions to Inline C, C++, Fortran Function Calls from R</td>
</tr>
<tr>
<td style="text-align: center;"><code>insight</code></td>
<td style="text-align: center;">0.17.1</td>
<td style="text-align: left;">Easy Access to Model Information for Various Model Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>interactiveDisplayBase</code></td>
<td style="text-align: center;">1.34.0</td>
<td style="text-align: left;">Base package for enabling powerful shiny web displays of Bioconductor objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>intervals</code></td>
<td style="text-align: center;">0.15.2</td>
<td style="text-align: left;">Tools for Working with Points and Intervals</td>
</tr>
<tr>
<td style="text-align: center;"><code>ipred</code></td>
<td style="text-align: center;">0.9-12</td>
<td style="text-align: left;">Improved Predictors</td>
</tr>
<tr>
<td style="text-align: center;"><code>IRanges</code></td>
<td style="text-align: center;">2.30.0</td>
<td style="text-align: left;">Foundation of integer range manipulation in Bioconductor</td>
</tr>
<tr>
<td style="text-align: center;"><code>ismev</code></td>
<td style="text-align: center;">1.42</td>
<td style="text-align: left;">An Introduction to Statistical Modeling of Extreme Values</td>
</tr>
<tr>
<td style="text-align: center;"><code>isoband</code></td>
<td style="text-align: center;">0.2.5</td>
<td style="text-align: left;">Generate Isolines and Isobands from Regularly Spaced Elevation Grids</td>
</tr>
<tr>
<td style="text-align: center;"><code>ISOcodes</code></td>
<td style="text-align: center;">2022.01.10</td>
<td style="text-align: left;">Selected ISO Codes</td>
</tr>
<tr>
<td style="text-align: center;"><code>isva</code></td>
<td style="text-align: center;">1.9</td>
<td style="text-align: left;">Independent Surrogate Variable Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>iterators</code></td>
<td style="text-align: center;">1.0.14</td>
<td style="text-align: left;">Provides Iterator Construct</td>
</tr>
<tr>
<td style="text-align: center;"><code>iterpc</code></td>
<td style="text-align: center;">0.4.2</td>
<td style="text-align: left;">Efficient Iterator for Permutations and Combinations</td>
</tr>
<tr>
<td style="text-align: center;"><code>JADE</code></td>
<td style="text-align: center;">2.0-3</td>
<td style="text-align: left;">Blind Source Separation Methods Based on Joint Diagonalization and Some BSS Performance Criteria</td>
</tr>
<tr>
<td style="text-align: center;"><code>jpeg</code></td>
<td style="text-align: center;">0.1-9</td>
<td style="text-align: left;">Read and write JPEG images</td>
</tr>
<tr>
<td style="text-align: center;"><code>jqr</code></td>
<td style="text-align: center;">1.2.3</td>
<td style="text-align: left;">Client for 'jq', a 'JSON' Processor</td>
</tr>
<tr>
<td style="text-align: center;"><code>jquerylib</code></td>
<td style="text-align: center;">0.1.4</td>
<td style="text-align: left;">Obtain 'jQuery' as an HTML Dependency Object</td>
</tr>
<tr>
<td style="text-align: center;"><code>jsonify</code></td>
<td style="text-align: center;">1.2.1</td>
<td style="text-align: left;">Convert Between 'R' Objects and Javascript Object Notation (JSON)</td>
</tr>
<tr>
<td style="text-align: center;"><code>jsonlite</code></td>
<td style="text-align: center;">1.8.0</td>
<td style="text-align: left;">A Simple and Robust JSON Parser and Generator for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>karyoploteR</code></td>
<td style="text-align: center;">1.22.0</td>
<td style="text-align: left;">Plot customizable linear genomes displaying arbitrary data</td>
</tr>
<tr>
<td style="text-align: center;"><code>KEGGREST</code></td>
<td style="text-align: center;">1.36.0</td>
<td style="text-align: left;">Client-side REST access to the Kyoto Encyclopedia of Genes and Genomes (KEGG)</td>
</tr>
<tr>
<td style="text-align: center;"><code>kernlab</code></td>
<td style="text-align: center;">0.9-30</td>
<td style="text-align: left;">Kernel-Based Machine Learning Lab</td>
</tr>
<tr>
<td style="text-align: center;"><code>KernSmooth</code></td>
<td style="text-align: center;">2.23-20</td>
<td style="text-align: left;">Functions for Kernel Smoothing Supporting Wand &amp; Jones (1995)</td>
</tr>
<tr>
<td style="text-align: center;"><code>knitr</code></td>
<td style="text-align: center;">1.39</td>
<td style="text-align: left;">A General-Purpose Package for Dynamic Report Generation in R</td>
</tr>
<tr>
<td style="text-align: center;"><code>kohonen</code></td>
<td style="text-align: center;">3.0.11</td>
<td style="text-align: left;">Supervised and Unsupervised Self-Organising Maps</td>
</tr>
<tr>
<td style="text-align: center;"><code>kpmt</code></td>
<td style="text-align: center;">0.1.0</td>
<td style="text-align: left;">Known Population Median Test</td>
</tr>
<tr>
<td style="text-align: center;"><code>labeling</code></td>
<td style="text-align: center;">0.4.2</td>
<td style="text-align: left;">Axis Labeling</td>
</tr>
<tr>
<td style="text-align: center;"><code>lambda.r</code></td>
<td style="text-align: center;">1.2.4</td>
<td style="text-align: left;">Modeling Data with Functional Programming</td>
</tr>
<tr>
<td style="text-align: center;"><code>later</code></td>
<td style="text-align: center;">1.3.0</td>
<td style="text-align: left;">Utilities for Scheduling Functions to Execute Later with Event Loops</td>
</tr>
<tr>
<td style="text-align: center;"><code>lattice</code></td>
<td style="text-align: center;">0.20-45</td>
<td style="text-align: left;">Trellis Graphics for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>latticeExtra</code></td>
<td style="text-align: center;">0.6-29</td>
<td style="text-align: left;">Extra Graphical Utilities Based on Lattice</td>
</tr>
<tr>
<td style="text-align: center;"><code>lava</code></td>
<td style="text-align: center;">1.6.10</td>
<td style="text-align: left;">Latent Variable Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>lazyeval</code></td>
<td style="text-align: center;">0.2.2</td>
<td style="text-align: left;">Lazy (Non-Standard) Evaluation</td>
</tr>
<tr>
<td style="text-align: center;"><code>lbfgsb3c</code></td>
<td style="text-align: center;">2020-3.2</td>
<td style="text-align: left;">Limited Memory BFGS Minimizer with Bounds on Parameters with optim() 'C' Interface</td>
</tr>
<tr>
<td style="text-align: center;"><code>leafem</code></td>
<td style="text-align: center;">0.2.0</td>
<td style="text-align: left;">'leaflet' Extensions for 'mapview'</td>
</tr>
<tr>
<td style="text-align: center;"><code>leaflet</code></td>
<td style="text-align: center;">2.1.1</td>
<td style="text-align: left;">Create Interactive Web Maps with the JavaScript 'Leaflet' Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>leaflet.providers</code></td>
<td style="text-align: center;">1.9.0</td>
<td style="text-align: left;">Leaflet Providers</td>
</tr>
<tr>
<td style="text-align: center;"><code>leafsync</code></td>
<td style="text-align: center;">0.1.0</td>
<td style="text-align: left;">Small Multiples for Leaflet Web Maps</td>
</tr>
<tr>
<td style="text-align: center;"><code>leaps</code></td>
<td style="text-align: center;">3.1</td>
<td style="text-align: left;">Regression Subset Selection</td>
</tr>
<tr>
<td style="text-align: center;"><code>learnr</code></td>
<td style="text-align: center;">0.10.1</td>
<td style="text-align: left;">Interactive Tutorials for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>lifecycle</code></td>
<td style="text-align: center;">1.0.1</td>
<td style="text-align: left;">Manage the Life Cycle of your Package Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>limma</code></td>
<td style="text-align: center;">3.52.1</td>
<td style="text-align: left;">Linear Models for Microarray Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>listenv</code></td>
<td style="text-align: center;">0.8.0</td>
<td style="text-align: left;">Environments Behaving (Almost) as Lists</td>
</tr>
<tr>
<td style="text-align: center;"><code>lme4</code></td>
<td style="text-align: center;">1.1-29</td>
<td style="text-align: left;">Linear Mixed-Effects Models using 'Eigen' and S4</td>
</tr>
<tr>
<td style="text-align: center;"><code>lmerTest</code></td>
<td style="text-align: center;">3.1-3</td>
<td style="text-align: left;">Tests in Linear Mixed Effects Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>lmom</code></td>
<td style="text-align: center;">2.9</td>
<td style="text-align: left;">L-Moments</td>
</tr>
<tr>
<td style="text-align: center;"><code>lmtest</code></td>
<td style="text-align: center;">0.9-40</td>
<td style="text-align: left;">Testing Linear Regression Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>locfit</code></td>
<td style="text-align: center;">1.5-9.5</td>
<td style="text-align: left;">Local Regression, Likelihood and Density Estimation</td>
</tr>
<tr>
<td style="text-align: center;"><code>longitudinal</code></td>
<td style="text-align: center;">1.1.13</td>
<td style="text-align: left;">Analysis of Multiple Time Course Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>loo</code></td>
<td style="text-align: center;">2.5.1</td>
<td style="text-align: left;">Efficient Leave-One-Out Cross-Validation and WAIC for Bayesian Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>lotri</code></td>
<td style="text-align: center;">0.4.2</td>
<td style="text-align: left;">A Simple Way to Specify Symmetric, Block Diagonal Matrices</td>
</tr>
<tr>
<td style="text-align: center;"><code>lpSolve</code></td>
<td style="text-align: center;">5.6.15</td>
<td style="text-align: left;">Interface to 'Lp_solve' v. 5.5 to Solve Linear/Integer Programs</td>
</tr>
<tr>
<td style="text-align: center;"><code>LSD</code></td>
<td style="text-align: center;">4.1-0</td>
<td style="text-align: left;">Lots of Superior Depictions</td>
</tr>
<tr>
<td style="text-align: center;"><code>ltm</code></td>
<td style="text-align: center;">1.2-0</td>
<td style="text-align: left;">Latent Trait Models under IRT</td>
</tr>
<tr>
<td style="text-align: center;"><code>lubridate</code></td>
<td style="text-align: center;">1.8.0</td>
<td style="text-align: left;">Make Dealing with Dates a Little Easier</td>
</tr>
<tr>
<td style="text-align: center;"><code>lumi</code></td>
<td style="text-align: center;">2.48.0</td>
<td style="text-align: left;">BeadArray Specific Methods for Illumina Methylation and Expression Microarrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>lwgeom</code></td>
<td style="text-align: center;">0.2-8</td>
<td style="text-align: left;">Bindings to Selected 'liblwgeom' Functions for Simple Features</td>
</tr>
<tr>
<td style="text-align: center;"><code>made4</code></td>
<td style="text-align: center;">1.70.0</td>
<td style="text-align: left;">Multivariate analysis of microarray data using ADE4</td>
</tr>
<tr>
<td style="text-align: center;"><code>magic</code></td>
<td style="text-align: center;">1.6-0</td>
<td style="text-align: left;">Create and Investigate Magic Squares</td>
</tr>
<tr>
<td style="text-align: center;"><code>magrittr</code></td>
<td style="text-align: center;">2.0.3</td>
<td style="text-align: left;">A Forward-Pipe Operator for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>maps</code></td>
<td style="text-align: center;">3.4.0</td>
<td style="text-align: left;">Draw Geographical Maps</td>
</tr>
<tr>
<td style="text-align: center;"><code>maptools</code></td>
<td style="text-align: center;">1.1-4</td>
<td style="text-align: left;">Tools for Handling Spatial Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>maptree</code></td>
<td style="text-align: center;">1.4-8</td>
<td style="text-align: left;">Mapping, Pruning, and Graphing Tree Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>markdown</code></td>
<td style="text-align: center;">1.1</td>
<td style="text-align: left;">Render Markdown with the C Library 'Sundown'</td>
</tr>
<tr>
<td style="text-align: center;"><code>marray</code></td>
<td style="text-align: center;">1.74.0</td>
<td style="text-align: left;">Exploratory analysis for two-color spotted microarray data</td>
</tr>
<tr>
<td style="text-align: center;"><code>MASS</code></td>
<td style="text-align: center;">7.3-56</td>
<td style="text-align: left;">Support Functions and Datasets for Venables and Ripley's MASS</td>
</tr>
<tr>
<td style="text-align: center;"><code>mathjaxr</code></td>
<td style="text-align: center;">1.6-0</td>
<td style="text-align: left;">Using 'Mathjax' in Rd Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>Matrix</code></td>
<td style="text-align: center;">1.5-3</td>
<td style="text-align: left;">Sparse and Dense Matrix Classes and Methods</td>
</tr>
<tr>
<td style="text-align: center;"><code>MatrixGenerics</code></td>
<td style="text-align: center;">1.8.0</td>
<td style="text-align: left;">S4 Generic Summary Statistic Functions that Operate on Matrix-Like Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>MatrixModels</code></td>
<td style="text-align: center;">0.5-0</td>
<td style="text-align: left;">Modelling with Sparse and Dense Matrices</td>
</tr>
<tr>
<td style="text-align: center;"><code>matrixStats</code></td>
<td style="text-align: center;">0.62.0</td>
<td style="text-align: left;">Functions that Apply to Rows and Columns of Matrices (and to Vectors)</td>
</tr>
<tr>
<td style="text-align: center;"><code>mclust</code></td>
<td style="text-align: center;">5.4.10</td>
<td style="text-align: left;">Gaussian Mixture Modelling for Model-Based Clustering, Classification, and Density Estimation</td>
</tr>
<tr>
<td style="text-align: center;"><code>mcmc</code></td>
<td style="text-align: center;">0.9-7</td>
<td style="text-align: left;">Markov Chain Monte Carlo</td>
</tr>
<tr>
<td style="text-align: center;"><code>memoise</code></td>
<td style="text-align: center;">2.0.1</td>
<td style="text-align: left;">'Memoisation' of Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>MendelianRandomization</code></td>
<td style="text-align: center;">0.6.0</td>
<td style="text-align: left;">Mendelian Randomization Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>metadat</code></td>
<td style="text-align: center;">1.2-0</td>
<td style="text-align: left;">Meta-Analysis Datasets</td>
</tr>
<tr>
<td style="text-align: center;"><code>metafor</code></td>
<td style="text-align: center;">3.4-0</td>
<td style="text-align: left;">Meta-Analysis Package for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>methods</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">Formal Methods and Classes</td>
</tr>
<tr>
<td style="text-align: center;"><code>methylumi</code></td>
<td style="text-align: center;">2.42.0</td>
<td style="text-align: left;">Handle Illumina methylation data</td>
</tr>
<tr>
<td style="text-align: center;"><code>mgcv</code></td>
<td style="text-align: center;">1.8-40</td>
<td style="text-align: left;">Mixed GAM Computation Vehicle with Automatic Smoothness Estimation</td>
</tr>
<tr>
<td style="text-align: center;"><code>mime</code></td>
<td style="text-align: center;">0.12</td>
<td style="text-align: left;">Map Filenames to MIME Types</td>
</tr>
<tr>
<td style="text-align: center;"><code>minfi</code></td>
<td style="text-align: center;">1.42.0</td>
<td style="text-align: left;">Analyze Illumina Infinium DNA methylation arrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>miniUI</code></td>
<td style="text-align: center;">0.1.1.1</td>
<td style="text-align: left;">Shiny UI Widgets for Small Screens</td>
</tr>
<tr>
<td style="text-align: center;"><code>minpack.lm</code></td>
<td style="text-align: center;">1.2-2</td>
<td style="text-align: left;">R Interface to the Levenberg-Marquardt Nonlinear Least-Squares Algorithm Found in MINPACK, Plus Support for Bounds</td>
</tr>
<tr>
<td style="text-align: center;"><code>minqa</code></td>
<td style="text-align: center;">1.2.4</td>
<td style="text-align: left;">Derivative-free optimization algorithms by quadratic approximation</td>
</tr>
<tr>
<td style="text-align: center;"><code>missMethyl</code></td>
<td style="text-align: center;">1.30.0</td>
<td style="text-align: left;">Analysing Illumina HumanMethylation BeadChip Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>mitools</code></td>
<td style="text-align: center;">2.4</td>
<td style="text-align: left;">Tools for Multiple Imputation of Missing Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>mlr</code></td>
<td style="text-align: center;">2.19.0</td>
<td style="text-align: left;">Machine Learning in R</td>
</tr>
<tr>
<td style="text-align: center;"><code>mnormt</code></td>
<td style="text-align: center;">2.0.2</td>
<td style="text-align: left;">The Multivariate Normal and t Distributions, and Their Truncated Versions</td>
</tr>
<tr>
<td style="text-align: center;"><code>ModelMetrics</code></td>
<td style="text-align: center;">1.2.2.2</td>
<td style="text-align: left;">Rapid Calculation of Model Metrics</td>
</tr>
<tr>
<td style="text-align: center;"><code>modelr</code></td>
<td style="text-align: center;">0.1.8</td>
<td style="text-align: left;">Modelling Functions that Work with the Pipe</td>
</tr>
<tr>
<td style="text-align: center;"><code>modeltools</code></td>
<td style="text-align: center;">0.2-23</td>
<td style="text-align: left;">Tools and Classes for Statistical Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>msm</code></td>
<td style="text-align: center;">1.6.9</td>
<td style="text-align: left;">Multi-State Markov and Hidden Markov Models in Continuous Time</td>
</tr>
<tr>
<td style="text-align: center;"><code>mstate</code></td>
<td style="text-align: center;">0.3.2</td>
<td style="text-align: left;">Data Preparation, Estimation and Prediction in Multi-State Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>multcomp</code></td>
<td style="text-align: center;">1.4-19</td>
<td style="text-align: left;">Simultaneous Inference in General Parametric Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>multtest</code></td>
<td style="text-align: center;">2.52.0</td>
<td style="text-align: left;">Resampling-based multiple hypothesis testing</td>
</tr>
<tr>
<td style="text-align: center;"><code>munsell</code></td>
<td style="text-align: center;">0.5.0</td>
<td style="text-align: left;">Utilities for Using Munsell Colours</td>
</tr>
<tr>
<td style="text-align: center;"><code>mvtnorm</code></td>
<td style="text-align: center;">1.1-3</td>
<td style="text-align: left;">Multivariate Normal and t Distributions</td>
</tr>
<tr>
<td style="text-align: center;"><code>n1qn1</code></td>
<td style="text-align: center;">6.0.1-11</td>
<td style="text-align: left;">Port of the 'Scilab' 'n1qn1' Module for Unconstrained BFGS Optimization</td>
</tr>
<tr>
<td style="text-align: center;"><code>nleqslv</code></td>
<td style="text-align: center;">3.3.2</td>
<td style="text-align: left;">Solve Systems of Nonlinear Equations</td>
</tr>
<tr>
<td style="text-align: center;"><code>nlme</code></td>
<td style="text-align: center;">3.1-157</td>
<td style="text-align: left;">Linear and Nonlinear Mixed Effects Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>nlmixr2</code></td>
<td style="text-align: center;">2.0.8</td>
<td style="text-align: left;">Nonlinear Mixed Effects Models in Population PK/PD</td>
</tr>
<tr>
<td style="text-align: center;"><code>nlmixr2data</code></td>
<td style="text-align: center;">2.0.7</td>
<td style="text-align: left;">Nonlinear Mixed Effects Models in Population PK/PD, Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>nlmixr2est</code></td>
<td style="text-align: center;">2.1.3</td>
<td style="text-align: left;">Nonlinear Mixed Effects Models in Population PK/PD, Estimation Routines</td>
</tr>
<tr>
<td style="text-align: center;"><code>nlmixr2extra</code></td>
<td style="text-align: center;">2.0.8</td>
<td style="text-align: left;">Nonlinear Mixed Effects Models in Population PK/PD, Extra Support Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>nlmixr2plot</code></td>
<td style="text-align: center;">2.0.7</td>
<td style="text-align: left;">Nonlinear Mixed Effects Models in Population PK/PD, Plot Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>nloptr</code></td>
<td style="text-align: center;">2.0.2</td>
<td style="text-align: left;">R Interface to NLopt</td>
</tr>
<tr>
<td style="text-align: center;"><code>NLP</code></td>
<td style="text-align: center;">0.2-1</td>
<td style="text-align: left;">Natural Language Processing Infrastructure</td>
</tr>
<tr>
<td style="text-align: center;"><code>nnet</code></td>
<td style="text-align: center;">7.3-17</td>
<td style="text-align: left;">Feed-Forward Neural Networks and Multinomial Log-Linear Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>nor1mix</code></td>
<td style="text-align: center;">1.3-0</td>
<td style="text-align: left;">Normal aka Gaussian (1-d) Mixture Models (S3 Classes and Methods)</td>
</tr>
<tr>
<td style="text-align: center;"><code>npde</code></td>
<td style="text-align: center;">3.2</td>
<td style="text-align: left;">Normalised Prediction Distribution Errors for Nonlinear Mixed-Effect Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>numDeriv</code></td>
<td style="text-align: center;">2016.8-1.1</td>
<td style="text-align: left;">Accurate Numerical Derivatives</td>
</tr>
<tr>
<td style="text-align: center;"><code>OPE</code></td>
<td style="text-align: center;">0.7</td>
<td style="text-align: left;">Outer-product emulator</td>
</tr>
<tr>
<td style="text-align: center;"><code>openssl</code></td>
<td style="text-align: center;">2.0.2</td>
<td style="text-align: left;">Toolkit for Encryption, Signatures and Certificates Based on OpenSSL</td>
</tr>
<tr>
<td style="text-align: center;"><code>optextras</code></td>
<td style="text-align: center;">2019-12.4</td>
<td style="text-align: left;">Tools to Support Optimization Possibly with Bounds and Masks</td>
</tr>
<tr>
<td style="text-align: center;"><code>optparse</code></td>
<td style="text-align: center;">1.7.1</td>
<td style="text-align: left;">Command Line Option Parser</td>
</tr>
<tr>
<td style="text-align: center;"><code>org.Hs.eg.db</code></td>
<td style="text-align: center;">3.15.0</td>
<td style="text-align: left;">Genome wide annotation for Human</td>
</tr>
<tr>
<td style="text-align: center;"><code>pander</code></td>
<td style="text-align: center;">0.6.5</td>
<td style="text-align: left;">An R 'Pandoc' Writer</td>
</tr>
<tr>
<td style="text-align: center;"><code>parallel</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">Support for Parallel computation in R</td>
</tr>
<tr>
<td style="text-align: center;"><code>parallelly</code></td>
<td style="text-align: center;">1.31.1</td>
<td style="text-align: left;">Enhancing the 'parallel' Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>parallelMap</code></td>
<td style="text-align: center;">1.5.1</td>
<td style="text-align: left;">Unified Interface to Parallelization Back-Ends</td>
</tr>
<tr>
<td style="text-align: center;"><code>ParamHelpers</code></td>
<td style="text-align: center;">1.14</td>
<td style="text-align: left;">Helpers for Parameters in Black-Box Optimization, Tuning and Machine Learning</td>
</tr>
<tr>
<td style="text-align: center;"><code>paran</code></td>
<td style="text-align: center;">1.5.2</td>
<td style="text-align: left;">Horn's Test of Principal Components/Factors</td>
</tr>
<tr>
<td style="text-align: center;"><code>pbapply</code></td>
<td style="text-align: center;">1.5-0</td>
<td style="text-align: left;">Adding Progress Bar to '*apply' Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>pbkrtest</code></td>
<td style="text-align: center;">0.5.1</td>
<td style="text-align: left;">Parametric Bootstrap, Kenward-Roger and Satterthwaite Based Methods for Test in Mixed Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>pcaPP</code></td>
<td style="text-align: center;">2.0-1</td>
<td style="text-align: left;">Robust PCA by Projection Pursuit</td>
</tr>
<tr>
<td style="text-align: center;"><code>pegas</code></td>
<td style="text-align: center;">1.1</td>
<td style="text-align: left;">Population and Evolutionary Genetics Analysis System</td>
</tr>
<tr>
<td style="text-align: center;"><code>permute</code></td>
<td style="text-align: center;">0.9-7</td>
<td style="text-align: left;">Functions for Generating Restricted Permutations of Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>phangorn</code></td>
<td style="text-align: center;">2.8.1</td>
<td style="text-align: left;">Phylogenetic Reconstruction and Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>phia</code></td>
<td style="text-align: center;">0.2-1</td>
<td style="text-align: left;">Post-Hoc Interaction Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>pillar</code></td>
<td style="text-align: center;">1.7.0</td>
<td style="text-align: left;">Coloured Formatting for Columns</td>
</tr>
<tr>
<td style="text-align: center;"><code>pixmap</code></td>
<td style="text-align: center;">0.4-12</td>
<td style="text-align: left;">Bitmap Images / Pixel Maps</td>
</tr>
<tr>
<td style="text-align: center;"><code>pkgbuild</code></td>
<td style="text-align: center;">1.3.1</td>
<td style="text-align: left;">Find Tools Needed to Build R Packages</td>
</tr>
<tr>
<td style="text-align: center;"><code>pkgconfig</code></td>
<td style="text-align: center;">2.0.3</td>
<td style="text-align: left;">Private Configuration for 'R' Packages</td>
</tr>
<tr>
<td style="text-align: center;"><code>pkgload</code></td>
<td style="text-align: center;">1.2.4</td>
<td style="text-align: left;">Simulate Package Installation and Attach</td>
</tr>
<tr>
<td style="text-align: center;"><code>pkgmaker</code></td>
<td style="text-align: center;">0.32.2</td>
<td style="text-align: left;">Development Utilities for R Packages</td>
</tr>
<tr>
<td style="text-align: center;"><code>plinkQC</code></td>
<td style="text-align: center;">0.3.4</td>
<td style="text-align: left;">Genotype Quality Control with 'PLINK'</td>
</tr>
<tr>
<td style="text-align: center;"><code>plogr</code></td>
<td style="text-align: center;">0.2.0</td>
<td style="text-align: left;">The 'plog' C++ Logging Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>plotly</code></td>
<td style="text-align: center;">4.10.0</td>
<td style="text-align: left;">Create Interactive Web Graphics via 'plotly.js'</td>
</tr>
<tr>
<td style="text-align: center;"><code>pls</code></td>
<td style="text-align: center;">2.8-0</td>
<td style="text-align: left;">Partial Least Squares and Principal Component Regression</td>
</tr>
<tr>
<td style="text-align: center;"><code>plyr</code></td>
<td style="text-align: center;">1.8.7</td>
<td style="text-align: left;">Tools for Splitting, Applying and Combining Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>png</code></td>
<td style="text-align: center;">0.1-7</td>
<td style="text-align: left;">Read and write PNG images</td>
</tr>
<tr>
<td style="text-align: center;"><code>poLCA</code></td>
<td style="text-align: center;">1.6.0.1</td>
<td style="text-align: left;">Polytomous Variable Latent Class Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>polspline</code></td>
<td style="text-align: center;">1.1.20</td>
<td style="text-align: left;">Polynomial Spline Routines</td>
</tr>
<tr>
<td style="text-align: center;"><code>polyclip</code></td>
<td style="text-align: center;">1.10-0</td>
<td style="text-align: left;">Polygon Clipping</td>
</tr>
<tr>
<td style="text-align: center;"><code>polycor</code></td>
<td style="text-align: center;">0.8-1</td>
<td style="text-align: left;">Polychoric and Polyserial Correlations</td>
</tr>
<tr>
<td style="text-align: center;"><code>posterior</code></td>
<td style="text-align: center;">1.2.1</td>
<td style="text-align: left;">Tools for Working with Posterior Distributions</td>
</tr>
<tr>
<td style="text-align: center;"><code>poweRlaw</code></td>
<td style="text-align: center;">0.70.6</td>
<td style="text-align: left;">Analysis of Heavy Tailed Distributions</td>
</tr>
<tr>
<td style="text-align: center;"><code>prabclus</code></td>
<td style="text-align: center;">2.3-2</td>
<td style="text-align: left;">Functions for Clustering and Testing of Presence-Absence, Abundance and Multilocus Genetic Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>pracma</code></td>
<td style="text-align: center;">2.3.8</td>
<td style="text-align: left;">Practical Numerical Math Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>praise</code></td>
<td style="text-align: center;">1.0.0</td>
<td style="text-align: left;">Praise Users</td>
</tr>
<tr>
<td style="text-align: center;"><code>PreciseSums</code></td>
<td style="text-align: center;">0.5</td>
<td style="text-align: left;">Accurate Floating Point Sums and Products</td>
</tr>
<tr>
<td style="text-align: center;"><code>preprocessCore</code></td>
<td style="text-align: center;">1.58.0</td>
<td style="text-align: left;">A collection of pre-processing functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>prettydoc</code></td>
<td style="text-align: center;">0.4.1</td>
<td style="text-align: left;">Creating Pretty Documents from R Markdown</td>
</tr>
<tr>
<td style="text-align: center;"><code>prettyunits</code></td>
<td style="text-align: center;">1.1.1</td>
<td style="text-align: left;">Pretty, Human Readable Formatting of Quantities</td>
</tr>
<tr>
<td style="text-align: center;"><code>pROC</code></td>
<td style="text-align: center;">1.18.0</td>
<td style="text-align: left;">Display and Analyze ROC Curves</td>
</tr>
<tr>
<td style="text-align: center;"><code>processx</code></td>
<td style="text-align: center;">3.5.3</td>
<td style="text-align: left;">Execute and Control System Processes</td>
</tr>
<tr>
<td style="text-align: center;"><code>prodlim</code></td>
<td style="text-align: center;">2019.11.13</td>
<td style="text-align: left;">Product-Limit Estimation for Censored Event History Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>profileModel</code></td>
<td style="text-align: center;">0.6.1</td>
<td style="text-align: left;">Profiling Inference Functions for Various Model Classes</td>
</tr>
<tr>
<td style="text-align: center;"><code>progress</code></td>
<td style="text-align: center;">1.2.2</td>
<td style="text-align: left;">Terminal Progress Bars</td>
</tr>
<tr>
<td style="text-align: center;"><code>progressr</code></td>
<td style="text-align: center;">0.10.0</td>
<td style="text-align: left;">An Inclusive, Unifying API for Progress Updates</td>
</tr>
<tr>
<td style="text-align: center;"><code>promises</code></td>
<td style="text-align: center;">1.2.0.1</td>
<td style="text-align: left;">Abstractions for Promise-Based Asynchronous Programming</td>
</tr>
<tr>
<td style="text-align: center;"><code>ProtGenerics</code></td>
<td style="text-align: center;">1.28.0</td>
<td style="text-align: left;">Generic infrastructure for Bioconductor mass spectrometry packages</td>
</tr>
<tr>
<td style="text-align: center;"><code>proto</code></td>
<td style="text-align: center;">1.0.0</td>
<td style="text-align: left;">Prototype Object-Based Programming</td>
</tr>
<tr>
<td style="text-align: center;"><code>protolite</code></td>
<td style="text-align: center;">2.1.1</td>
<td style="text-align: left;">Highly Optimized Protocol Buffer Serializers</td>
</tr>
<tr>
<td style="text-align: center;"><code>proxy</code></td>
<td style="text-align: center;">0.4-26</td>
<td style="text-align: left;">Distance and Similarity Measures</td>
</tr>
<tr>
<td style="text-align: center;"><code>ps</code></td>
<td style="text-align: center;">1.7.0</td>
<td style="text-align: left;">List, Query, Manipulate System Processes</td>
</tr>
<tr>
<td style="text-align: center;"><code>PSCBS</code></td>
<td style="text-align: center;">0.66.0</td>
<td style="text-align: left;">Analysis of Parent-Specific DNA Copy Numbers</td>
</tr>
<tr>
<td style="text-align: center;"><code>pspline</code></td>
<td style="text-align: center;">1.0-19</td>
<td style="text-align: left;">Penalized Smoothing Splines</td>
</tr>
<tr>
<td style="text-align: center;"><code>psych</code></td>
<td style="text-align: center;">2.2.5</td>
<td style="text-align: left;">Procedures for Psychological, Psychometric, and Personality Research</td>
</tr>
<tr>
<td style="text-align: center;"><code>purrr</code></td>
<td style="text-align: center;">0.3.4</td>
<td style="text-align: left;">Functional Programming Tools</td>
</tr>
<tr>
<td style="text-align: center;"><code>qs</code></td>
<td style="text-align: center;">0.25.4</td>
<td style="text-align: left;">Quick Serialization of R Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>quadprog</code></td>
<td style="text-align: center;">1.5-8</td>
<td style="text-align: left;">Functions to Solve Quadratic Programming Problems</td>
</tr>
<tr>
<td style="text-align: center;"><code>quanteda</code></td>
<td style="text-align: center;">3.2.1</td>
<td style="text-align: left;">Quantitative Analysis of Textual Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>quanteda.corpora</code></td>
<td style="text-align: center;">0.9.2</td>
<td style="text-align: left;">A collection of corpora for quanteda</td>
</tr>
<tr>
<td style="text-align: center;"><code>quantmod</code></td>
<td style="text-align: center;">0.4.20</td>
<td style="text-align: left;">Quantitative Financial Modelling Framework</td>
</tr>
<tr>
<td style="text-align: center;"><code>quantreg</code></td>
<td style="text-align: center;">5.93</td>
<td style="text-align: left;">Quantile Regression</td>
</tr>
<tr>
<td style="text-align: center;"><code>qvalue</code></td>
<td style="text-align: center;">2.28.0</td>
<td style="text-align: left;">Q-value estimation for false discovery rate control</td>
</tr>
<tr>
<td style="text-align: center;"><code>qvcalc</code></td>
<td style="text-align: center;">1.0.2</td>
<td style="text-align: left;">Quasi Variances for Factor Effects in Statistical Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>R.cache</code></td>
<td style="text-align: center;">0.15.0</td>
<td style="text-align: left;">Fast and Light-Weight Caching (Memoization) of Objects and Results to Speed Up Computations</td>
</tr>
<tr>
<td style="text-align: center;"><code>R.devices</code></td>
<td style="text-align: center;">2.17.0</td>
<td style="text-align: left;">Unified Handling of Graphics Devices</td>
</tr>
<tr>
<td style="text-align: center;"><code>R.filesets</code></td>
<td style="text-align: center;">2.14.0</td>
<td style="text-align: left;">Easy Handling of and Access to Files Organized in Structured Directories</td>
</tr>
<tr>
<td style="text-align: center;"><code>R.huge</code></td>
<td style="text-align: center;">0.9.0</td>
<td style="text-align: left;">Methods for Accessing Huge Amounts of Data [deprecated]</td>
</tr>
<tr>
<td style="text-align: center;"><code>R.methodsS3</code></td>
<td style="text-align: center;">1.8.1</td>
<td style="text-align: left;">S3 Methods Simplified</td>
</tr>
<tr>
<td style="text-align: center;"><code>R.oo</code></td>
<td style="text-align: center;">1.24.0</td>
<td style="text-align: left;">R Object-Oriented Programming with or without References</td>
</tr>
<tr>
<td style="text-align: center;"><code>R.rsp</code></td>
<td style="text-align: center;">0.44.0</td>
<td style="text-align: left;">Dynamic Generation of Scientific Reports</td>
</tr>
<tr>
<td style="text-align: center;"><code>R.utils</code></td>
<td style="text-align: center;">2.11.0</td>
<td style="text-align: left;">Various Programming Utilities</td>
</tr>
<tr>
<td style="text-align: center;"><code>R2HTML</code></td>
<td style="text-align: center;">2.3.3</td>
<td style="text-align: left;">HTML Exportation for R Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>R2WinBUGS</code></td>
<td style="text-align: center;">2.1-21</td>
<td style="text-align: left;">Running 'WinBUGS' and 'OpenBUGS' from 'R' / 'S-PLUS'</td>
</tr>
<tr>
<td style="text-align: center;"><code>R6</code></td>
<td style="text-align: center;">2.5.1</td>
<td style="text-align: left;">Encapsulated Classes with Reference Semantics</td>
</tr>
<tr>
<td style="text-align: center;"><code>randomForest</code></td>
<td style="text-align: center;">4.7-1.1</td>
<td style="text-align: left;">Breiman and Cutler's Random Forests for Classification and Regression</td>
</tr>
<tr>
<td style="text-align: center;"><code>rapidjsonr</code></td>
<td style="text-align: center;">1.2.0</td>
<td style="text-align: left;">'Rapidjson' C++ Header Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>RApiSerialize</code></td>
<td style="text-align: center;">0.1.2</td>
<td style="text-align: left;">R API Serialization</td>
</tr>
<tr>
<td style="text-align: center;"><code>rappdirs</code></td>
<td style="text-align: center;">0.3.3</td>
<td style="text-align: left;">Application Directories: Determine Where to Save Data, Caches, and Logs</td>
</tr>
<tr>
<td style="text-align: center;"><code>raster</code></td>
<td style="text-align: center;">3.5-15</td>
<td style="text-align: left;">Geographic Data Analysis and Modeling</td>
</tr>
<tr>
<td style="text-align: center;"><code>RBGL</code></td>
<td style="text-align: center;">1.72.0</td>
<td style="text-align: left;">An interface to the BOOST graph library</td>
</tr>
<tr>
<td style="text-align: center;"><code>rcmdcheck</code></td>
<td style="text-align: center;">1.4.0</td>
<td style="text-align: left;">Run 'R CMD check' from 'R' and Capture Results</td>
</tr>
<tr>
<td style="text-align: center;"><code>RColorBrewer</code></td>
<td style="text-align: center;">1.1-3</td>
<td style="text-align: left;">ColorBrewer Palettes</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rcpp</code></td>
<td style="text-align: center;">1.0.9</td>
<td style="text-align: left;">Seamless R and C++ Integration</td>
</tr>
<tr>
<td style="text-align: center;"><code>RcppArmadillo</code></td>
<td style="text-align: center;">0.11.4.2.1</td>
<td style="text-align: left;">'Rcpp' Integration for the 'Armadillo' Templated Linear Algebra Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>RcppEigen</code></td>
<td style="text-align: center;">0.3.3.9.2</td>
<td style="text-align: left;">'Rcpp' Integration for the 'Eigen' Templated Linear Algebra Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>RcppGSL</code></td>
<td style="text-align: center;">0.3.11</td>
<td style="text-align: left;">'Rcpp' Integration for 'GNU GSL' Vectors and Matrices</td>
</tr>
<tr>
<td style="text-align: center;"><code>RcppParallel</code></td>
<td style="text-align: center;">5.1.5</td>
<td style="text-align: left;">Parallel Programming Tools for 'Rcpp'</td>
</tr>
<tr>
<td style="text-align: center;"><code>RcppZiggurat</code></td>
<td style="text-align: center;">0.1.6</td>
<td style="text-align: left;">'Rcpp' Integration of Different "Ziggurat" Normal RNG Implementations</td>
</tr>
<tr>
<td style="text-align: center;"><code>RCurl</code></td>
<td style="text-align: center;">1.98-1.6</td>
<td style="text-align: left;">General Network (HTTP/FTP/...) Client Interface for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>readr</code></td>
<td style="text-align: center;">2.1.2</td>
<td style="text-align: left;">Read Rectangular Text Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>readxl</code></td>
<td style="text-align: center;">1.4.0</td>
<td style="text-align: left;">Read Excel Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>recipes</code></td>
<td style="text-align: center;">0.2.0</td>
<td style="text-align: left;">Preprocessing and Feature Engineering Steps for Modeling</td>
</tr>
<tr>
<td style="text-align: center;"><code>regioneR</code></td>
<td style="text-align: center;">1.28.0</td>
<td style="text-align: left;">Association analysis of genomic regions based on permutation tests</td>
</tr>
<tr>
<td style="text-align: center;"><code>registry</code></td>
<td style="text-align: center;">0.5-1</td>
<td style="text-align: left;">Infrastructure for R Package Registries</td>
</tr>
<tr>
<td style="text-align: center;"><code>relaimpo</code></td>
<td style="text-align: center;">2.2-6</td>
<td style="text-align: left;">Relative Importance of Regressors in Linear Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>rematch</code></td>
<td style="text-align: center;">1.0.1</td>
<td style="text-align: left;">Match Regular Expressions with a Nicer 'API'</td>
</tr>
<tr>
<td style="text-align: center;"><code>rematch2</code></td>
<td style="text-align: center;">2.1.2</td>
<td style="text-align: left;">Tidy Output from Regular Expression Matching</td>
</tr>
<tr>
<td style="text-align: center;"><code>remotes</code></td>
<td style="text-align: center;">2.4.2</td>
<td style="text-align: left;">R Package Installation from Remote Repositories, Including 'GitHub'</td>
</tr>
<tr>
<td style="text-align: center;"><code>renv</code></td>
<td style="text-align: center;">0.15.4</td>
<td style="text-align: left;">Project Environments</td>
</tr>
<tr>
<td style="text-align: center;"><code>Repitools</code></td>
<td style="text-align: center;">1.42.0</td>
<td style="text-align: left;">Epigenomic tools</td>
</tr>
<tr>
<td style="text-align: center;"><code>reprex</code></td>
<td style="text-align: center;">2.0.1</td>
<td style="text-align: left;">Prepare Reproducible Example Code via the Clipboard</td>
</tr>
<tr>
<td style="text-align: center;"><code>reshape</code></td>
<td style="text-align: center;">0.8.9</td>
<td style="text-align: left;">Flexibly Reshape Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>reshape2</code></td>
<td style="text-align: center;">1.4.4</td>
<td style="text-align: left;">Flexibly Reshape Data: A Reboot of the Reshape Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>restfulr</code></td>
<td style="text-align: center;">0.0.13</td>
<td style="text-align: left;">R Interface to RESTful Web Services</td>
</tr>
<tr>
<td style="text-align: center;"><code>rex</code></td>
<td style="text-align: center;">1.2.1</td>
<td style="text-align: left;">Friendly Regular Expressions</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rfast</code></td>
<td style="text-align: center;">2.0.6</td>
<td style="text-align: left;">A Collection of Efficient and Extremely Fast R Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>rgdal</code></td>
<td style="text-align: center;">1.5-32</td>
<td style="text-align: left;">Bindings for the 'Geospatial' Data Abstraction Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>rgeos</code></td>
<td style="text-align: center;">0.5-9</td>
<td style="text-align: left;">Interface to Geometry Engine - Open Source ('GEOS')</td>
</tr>
<tr>
<td style="text-align: center;"><code>rgl</code></td>
<td style="text-align: center;">0.108.3.2</td>
<td style="text-align: left;">3D Visualization Using OpenGL</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rgraphviz</code></td>
<td style="text-align: center;">2.40.0</td>
<td style="text-align: left;">Provides plotting capabilities for R graph objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>rhdf5</code></td>
<td style="text-align: center;">2.40.0</td>
<td style="text-align: left;">R Interface to HDF5</td>
</tr>
<tr>
<td style="text-align: center;"><code>rhdf5filters</code></td>
<td style="text-align: center;">1.8.0</td>
<td style="text-align: left;">HDF5 Compression Filters</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rhdf5lib</code></td>
<td style="text-align: center;">1.18.2</td>
<td style="text-align: left;">hdf5 library as an R package</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rhtslib</code></td>
<td style="text-align: center;">1.28.0</td>
<td style="text-align: left;">HTSlib high-throughput sequencing library as an R package</td>
</tr>
<tr>
<td style="text-align: center;"><code>Ringo</code></td>
<td style="text-align: center;">1.60.0</td>
<td style="text-align: left;">R Investigation of ChIP-chip Oligoarrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>rjson</code></td>
<td style="text-align: center;">0.2.21</td>
<td style="text-align: left;">JSON for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>RJSONIO</code></td>
<td style="text-align: center;">1.3-1.6</td>
<td style="text-align: left;">Serialize R Objects to JSON, JavaScript Object Notation</td>
</tr>
<tr>
<td style="text-align: center;"><code>rlang</code></td>
<td style="text-align: center;">1.0.2</td>
<td style="text-align: left;">Functions for Base Types and Core R and 'Tidyverse' Features</td>
</tr>
<tr>
<td style="text-align: center;"><code>rlecuyer</code></td>
<td style="text-align: center;">0.3-5</td>
<td style="text-align: left;">R Interface to RNG with Multiple Streams</td>
</tr>
<tr>
<td style="text-align: center;"><code>rlist</code></td>
<td style="text-align: center;">0.4.6.2</td>
<td style="text-align: left;">A Toolbox for Non-Tabular Data Manipulation</td>
</tr>
<tr>
<td style="text-align: center;"><code>rmarkdown</code></td>
<td style="text-align: center;">2.14</td>
<td style="text-align: left;">Dynamic Documents for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rmpfr</code></td>
<td style="text-align: center;">0.8-7</td>
<td style="text-align: left;">R MPFR - Multiple Precision Floating-Point Reliable</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rmpi</code></td>
<td style="text-align: center;">0.6-9.2</td>
<td style="text-align: left;">Interface (Wrapper) to MPI (Message-Passing Interface)</td>
</tr>
<tr>
<td style="text-align: center;"><code>rms</code></td>
<td style="text-align: center;">6.3-0</td>
<td style="text-align: left;">Regression Modeling Strategies</td>
</tr>
<tr>
<td style="text-align: center;"><code>RMySQL</code></td>
<td style="text-align: center;">0.10.23</td>
<td style="text-align: left;">Database Interface and 'MySQL' Driver for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>RNetCDF</code></td>
<td style="text-align: center;">2.5-2</td>
<td style="text-align: left;">Interface to 'NetCDF' Datasets</td>
</tr>
<tr>
<td style="text-align: center;"><code>rngtools</code></td>
<td style="text-align: center;">1.5.2</td>
<td style="text-align: left;">Utility Functions for Working with Random Number Generators</td>
</tr>
<tr>
<td style="text-align: center;"><code>robustbase</code></td>
<td style="text-align: center;">0.95-0</td>
<td style="text-align: left;">Basic Robust Statistics</td>
</tr>
<tr>
<td style="text-align: center;"><code>ROC</code></td>
<td style="text-align: center;">1.72.0</td>
<td style="text-align: left;">utilities for ROC, with microarray focus</td>
</tr>
<tr>
<td style="text-align: center;"><code>rootSolve</code></td>
<td style="text-align: center;">1.8.2.3</td>
<td style="text-align: left;">Nonlinear Root Finding, Equilibrium and Steady-State Analysis of Ordinary Differential Equations</td>
</tr>
<tr>
<td style="text-align: center;"><code>roxygen2</code></td>
<td style="text-align: center;">7.2.0</td>
<td style="text-align: left;">In-Line Documentation for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>rpart</code></td>
<td style="text-align: center;">4.1.16</td>
<td style="text-align: left;">Recursive Partitioning and Regression Trees</td>
</tr>
<tr>
<td style="text-align: center;"><code>RPMM</code></td>
<td style="text-align: center;">1.25</td>
<td style="text-align: left;">Recursively Partitioned Mixture Model</td>
</tr>
<tr>
<td style="text-align: center;"><code>rprojroot</code></td>
<td style="text-align: center;">2.0.3</td>
<td style="text-align: left;">Finding Files in Project Subdirectories</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rsamtools</code></td>
<td style="text-align: center;">2.12.0</td>
<td style="text-align: left;">Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rsolnp</code></td>
<td style="text-align: center;">1.16</td>
<td style="text-align: left;">General Non-Linear Optimization</td>
</tr>
<tr>
<td style="text-align: center;"><code>RSQLite</code></td>
<td style="text-align: center;">2.2.14</td>
<td style="text-align: left;">SQLite Interface for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>rstan</code></td>
<td style="text-align: center;">2.21.5</td>
<td style="text-align: left;">R Interface to Stan</td>
</tr>
<tr>
<td style="text-align: center;"><code>rstanarm</code></td>
<td style="text-align: center;">2.21.3</td>
<td style="text-align: left;">Bayesian Applied Regression Modeling via Stan</td>
</tr>
<tr>
<td style="text-align: center;"><code>rstantools</code></td>
<td style="text-align: center;">2.2.0</td>
<td style="text-align: left;">Tools for Developing R Packages Interfacing with 'Stan'</td>
</tr>
<tr>
<td style="text-align: center;"><code>rstudioapi</code></td>
<td style="text-align: center;">0.13</td>
<td style="text-align: left;">Safely Access the RStudio API</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rsubread</code></td>
<td style="text-align: center;">2.10.1</td>
<td style="text-align: left;">Mapping, quantification and variant analysis of sequencing data</td>
</tr>
<tr>
<td style="text-align: center;"><code>rtracklayer</code></td>
<td style="text-align: center;">1.56.0</td>
<td style="text-align: left;">R interface to genome annotation files and the UCSC genome browser</td>
</tr>
<tr>
<td style="text-align: center;"><code>ruv</code></td>
<td style="text-align: center;">0.9.7.1</td>
<td style="text-align: left;">Detect and Remove Unwanted Variation using Negative Controls</td>
</tr>
<tr>
<td style="text-align: center;"><code>rversions</code></td>
<td style="text-align: center;">2.1.1</td>
<td style="text-align: left;">Query 'R' Versions, Including 'r-release' and 'r-oldrel'</td>
</tr>
<tr>
<td style="text-align: center;"><code>rvest</code></td>
<td style="text-align: center;">1.0.2</td>
<td style="text-align: left;">Easily Harvest (Scrape) Web Pages</td>
</tr>
<tr>
<td style="text-align: center;"><code>Rvmmin</code></td>
<td style="text-align: center;">2018-4.17.1</td>
<td style="text-align: left;">Variable Metric Nonlinear Function Minimization</td>
</tr>
<tr>
<td style="text-align: center;"><code>rxode2</code></td>
<td style="text-align: center;">2.0.11</td>
<td style="text-align: left;">Facilities for Simulating from ODE-Based Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>rxode2et</code></td>
<td style="text-align: center;">2.0.9</td>
<td style="text-align: left;">Event Table Functions for 'rxode2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>rxode2ll</code></td>
<td style="text-align: center;">2.0.9</td>
<td style="text-align: left;">Log-Likelihood Functions for 'rxode2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>rxode2parse</code></td>
<td style="text-align: center;">2.0.13</td>
<td style="text-align: left;">Parsing and Code Generation Functions for 'rxode2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>rxode2random</code></td>
<td style="text-align: center;">2.0.9</td>
<td style="text-align: left;">Random Number Generation Functions for 'rxode2'</td>
</tr>
<tr>
<td style="text-align: center;"><code>s2</code></td>
<td style="text-align: center;">1.0.7</td>
<td style="text-align: left;">Spherical Geometry Operators Using the S2 Geometry Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>S4Vectors</code></td>
<td style="text-align: center;">0.34.0</td>
<td style="text-align: left;">Foundation of vector-like and list-like containers in Bioconductor</td>
</tr>
<tr>
<td style="text-align: center;"><code>saemix</code></td>
<td style="text-align: center;">3.0</td>
<td style="text-align: left;">Stochastic Approximation Expectation Maximization (SAEM) Algorithm</td>
</tr>
<tr>
<td style="text-align: center;"><code>sampling</code></td>
<td style="text-align: center;">2.9</td>
<td style="text-align: left;">Survey Sampling</td>
</tr>
<tr>
<td style="text-align: center;"><code>sandwich</code></td>
<td style="text-align: center;">3.0-1</td>
<td style="text-align: left;">Robust Covariance Matrix Estimators</td>
</tr>
<tr>
<td style="text-align: center;"><code>sass</code></td>
<td style="text-align: center;">0.4.1</td>
<td style="text-align: left;">Syntactically Awesome Style Sheets ('Sass')</td>
</tr>
<tr>
<td style="text-align: center;"><code>scales</code></td>
<td style="text-align: center;">1.2.0</td>
<td style="text-align: left;">Scale Functions for Visualization</td>
</tr>
<tr>
<td style="text-align: center;"><code>scam</code></td>
<td style="text-align: center;">1.2-12</td>
<td style="text-align: left;">Shape Constrained Additive Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>scatterplot3d</code></td>
<td style="text-align: center;">0.3-41</td>
<td style="text-align: left;">3D Scatter Plot</td>
</tr>
<tr>
<td style="text-align: center;"><code>schoolmath</code></td>
<td style="text-align: center;">0.4.1</td>
<td style="text-align: left;">Functions and Datasets for Math Used in School</td>
</tr>
<tr>
<td style="text-align: center;"><code>scrime</code></td>
<td style="text-align: center;">1.3.5</td>
<td style="text-align: left;">Analysis of High-Dimensional Categorical Data Such as SNP Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>segmented</code></td>
<td style="text-align: center;">1.5-0</td>
<td style="text-align: left;">Regression Models with Break-Points / Change-Points Estimation</td>
</tr>
<tr>
<td style="text-align: center;"><code>selectr</code></td>
<td style="text-align: center;">0.4-2</td>
<td style="text-align: left;">Translate CSS Selectors to XPath Expressions</td>
</tr>
<tr>
<td style="text-align: center;"><code>sendmailR</code></td>
<td style="text-align: center;">1.2-1</td>
<td style="text-align: left;">send email using R</td>
</tr>
<tr>
<td style="text-align: center;"><code>seqinr</code></td>
<td style="text-align: center;">4.2-16</td>
<td style="text-align: left;">Biological Sequences Retrieval and Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>seqLogo</code></td>
<td style="text-align: center;">1.62.0</td>
<td style="text-align: left;">Sequence logos for DNA sequence alignments</td>
</tr>
<tr>
<td style="text-align: center;"><code>sessioninfo</code></td>
<td style="text-align: center;">1.2.2</td>
<td style="text-align: left;">R Session Information</td>
</tr>
<tr>
<td style="text-align: center;"><code>sf</code></td>
<td style="text-align: center;">1.0-7</td>
<td style="text-align: left;">Simple Features for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>sfheaders</code></td>
<td style="text-align: center;">0.4.0</td>
<td style="text-align: left;">Converts Between R Objects and Simple Feature Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>sfsmisc</code></td>
<td style="text-align: center;">1.1-13</td>
<td style="text-align: left;">Utilities from 'Seminar fuer Statistik' ETH Zurich</td>
</tr>
<tr>
<td style="text-align: center;"><code>shape</code></td>
<td style="text-align: center;">1.4.6</td>
<td style="text-align: left;">Functions for Plotting Graphical Shapes, Colors</td>
</tr>
<tr>
<td style="text-align: center;"><code>shiny</code></td>
<td style="text-align: center;">1.7.1</td>
<td style="text-align: left;">Web Application Framework for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>shinyjs</code></td>
<td style="text-align: center;">2.1.0</td>
<td style="text-align: left;">Easily Improve the User Experience of Your Shiny Apps in Seconds</td>
</tr>
<tr>
<td style="text-align: center;"><code>shinystan</code></td>
<td style="text-align: center;">2.6.0</td>
<td style="text-align: left;">Interactive Visual and Numerical Diagnostics and Posterior Analysis for Bayesian Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>shinythemes</code></td>
<td style="text-align: center;">1.2.0</td>
<td style="text-align: left;">Themes for Shiny</td>
</tr>
<tr>
<td style="text-align: center;"><code>ShortRead</code></td>
<td style="text-align: center;">1.54.0</td>
<td style="text-align: left;">FASTQ input and manipulation</td>
</tr>
<tr>
<td style="text-align: center;"><code>siggenes</code></td>
<td style="text-align: center;">1.70.0</td>
<td style="text-align: left;">Multiple Testing using SAM and Efron's Empirical Bayes Approaches</td>
</tr>
<tr>
<td style="text-align: center;"><code>sitmo</code></td>
<td style="text-align: center;">2.0.2</td>
<td style="text-align: left;">Parallel Pseudo Random Number Generator (PPRNG) 'sitmo' Header Files</td>
</tr>
<tr>
<td style="text-align: center;"><code>slam</code></td>
<td style="text-align: center;">0.1-50</td>
<td style="text-align: left;">Sparse Lightweight Arrays and Matrices</td>
</tr>
<tr>
<td style="text-align: center;"><code>sn</code></td>
<td style="text-align: center;">2.0.2</td>
<td style="text-align: left;">The Skew-Normal and Related Distributions Such as the Skew-t and the SUN</td>
</tr>
<tr>
<td style="text-align: center;"><code>snow</code></td>
<td style="text-align: center;">0.4-4</td>
<td style="text-align: left;">Simple Network of Workstations</td>
</tr>
<tr>
<td style="text-align: center;"><code>SnowballC</code></td>
<td style="text-align: center;">0.7.0</td>
<td style="text-align: left;">Snowball Stemmers Based on the C 'libstemmer' UTF-8 Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>snowfall</code></td>
<td style="text-align: center;">1.84-6.1</td>
<td style="text-align: left;">Easier cluster computing (based on snow).</td>
</tr>
<tr>
<td style="text-align: center;"><code>softImpute</code></td>
<td style="text-align: center;">1.4-1</td>
<td style="text-align: left;">Matrix Completion via Iterative Soft-Thresholded SVD</td>
</tr>
<tr>
<td style="text-align: center;"><code>sourcetools</code></td>
<td style="text-align: center;">0.1.7</td>
<td style="text-align: left;">Tools for Reading, Tokenizing and Parsing R Code</td>
</tr>
<tr>
<td style="text-align: center;"><code>sp</code></td>
<td style="text-align: center;">1.4-7</td>
<td style="text-align: left;">Classes and Methods for Spatial Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>spam</code></td>
<td style="text-align: center;">2.8-0</td>
<td style="text-align: left;">SPArse Matrix</td>
</tr>
<tr>
<td style="text-align: center;"><code>SparseM</code></td>
<td style="text-align: center;">1.81</td>
<td style="text-align: left;">Sparse Linear Algebra</td>
</tr>
<tr>
<td style="text-align: center;"><code>sparseMatrixStats</code></td>
<td style="text-align: center;">1.8.0</td>
<td style="text-align: left;">Summary Statistics for Rows and Columns of Sparse Matrices</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatial</code></td>
<td style="text-align: center;">7.3-15</td>
<td style="text-align: left;">Functions for Kriging and Point Pattern Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatstat</code></td>
<td style="text-align: center;">2.3-4</td>
<td style="text-align: left;">Spatial Point Pattern Analysis, Model-Fitting, Simulation, Tests</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatstat.core</code></td>
<td style="text-align: center;">2.4-4</td>
<td style="text-align: left;">Core Functionality of the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatstat.data</code></td>
<td style="text-align: center;">2.2-0</td>
<td style="text-align: left;">Datasets for 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatstat.geom</code></td>
<td style="text-align: center;">2.4-0</td>
<td style="text-align: left;">Geometrical Functionality of the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatstat.linnet</code></td>
<td style="text-align: center;">2.3-2</td>
<td style="text-align: left;">Linear Networks Functionality of the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatstat.random</code></td>
<td style="text-align: center;">2.2-0</td>
<td style="text-align: left;">Random Generation Functionality for the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatstat.sparse</code></td>
<td style="text-align: center;">2.1-1</td>
<td style="text-align: left;">Sparse Three-Dimensional Arrays and Linear Algebra Utilities</td>
</tr>
<tr>
<td style="text-align: center;"><code>spatstat.utils</code></td>
<td style="text-align: center;">2.3-1</td>
<td style="text-align: left;">Utility Functions for 'spatstat'</td>
</tr>
<tr>
<td style="text-align: center;"><code>spData</code></td>
<td style="text-align: center;">2.0.1</td>
<td style="text-align: left;">Datasets for Spatial Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>spdep</code></td>
<td style="text-align: center;">1.2-4</td>
<td style="text-align: left;">Spatial Dependence: Weighting Schemes, Statistics</td>
</tr>
<tr>
<td style="text-align: center;"><code>splines</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">Regression Spline Functions and Classes</td>
</tr>
<tr>
<td style="text-align: center;"><code>SQUAREM</code></td>
<td style="text-align: center;">2021.1</td>
<td style="text-align: left;">Squared Extrapolation Methods for Accelerating EM-Like Monotone Algorithms</td>
</tr>
<tr>
<td style="text-align: center;"><code>stabledist</code></td>
<td style="text-align: center;">0.7-1</td>
<td style="text-align: left;">Stable Distribution Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>StanHeaders</code></td>
<td style="text-align: center;">2.21.0-7</td>
<td style="text-align: left;">C++ Header Files for Stan</td>
</tr>
<tr>
<td style="text-align: center;"><code>stars</code></td>
<td style="text-align: center;">0.5-5</td>
<td style="text-align: left;">Spatiotemporal Arrays, Raster and Vector Data Cubes</td>
</tr>
<tr>
<td style="text-align: center;"><code>startupmsg</code></td>
<td style="text-align: center;">0.9.6</td>
<td style="text-align: left;">Utilities for Start-Up Messages</td>
</tr>
<tr>
<td style="text-align: center;"><code>statmod</code></td>
<td style="text-align: center;">1.4.36</td>
<td style="text-align: left;">Statistical Modeling</td>
</tr>
<tr>
<td style="text-align: center;"><code>stats</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">The R Stats Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>stats4</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">Statistical Functions using S4 Classes</td>
</tr>
<tr>
<td style="text-align: center;"><code>stopwords</code></td>
<td style="text-align: center;">2.3</td>
<td style="text-align: left;">Multilingual Stopword Lists</td>
</tr>
<tr>
<td style="text-align: center;"><code>stringdist</code></td>
<td style="text-align: center;">0.9.8</td>
<td style="text-align: left;">Approximate String Matching, Fuzzy Text Search, and String Distance Functions</td>
</tr>
<tr>
<td style="text-align: center;"><code>stringfish</code></td>
<td style="text-align: center;">0.15.7</td>
<td style="text-align: left;">Alt String Implementation</td>
</tr>
<tr>
<td style="text-align: center;"><code>stringi</code></td>
<td style="text-align: center;">1.7.6</td>
<td style="text-align: left;">Character String Processing Facilities</td>
</tr>
<tr>
<td style="text-align: center;"><code>stringr</code></td>
<td style="text-align: center;">1.4.0</td>
<td style="text-align: left;">Simple, Consistent Wrappers for Common String Operations</td>
</tr>
<tr>
<td style="text-align: center;"><code>SummarizedExperiment</code></td>
<td style="text-align: center;">1.26.1</td>
<td style="text-align: left;">SummarizedExperiment container</td>
</tr>
<tr>
<td style="text-align: center;"><code>survey</code></td>
<td style="text-align: center;">4.1-1</td>
<td style="text-align: left;">Analysis of Complex Survey Samples</td>
</tr>
<tr>
<td style="text-align: center;"><code>survival</code></td>
<td style="text-align: center;">3.3-1</td>
<td style="text-align: left;">Survival Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>sva</code></td>
<td style="text-align: center;">3.44.0</td>
<td style="text-align: left;">Surrogate Variable Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>symengine</code></td>
<td style="text-align: center;">0.2.2</td>
<td style="text-align: left;">Interface to the 'SymEngine' Library</td>
</tr>
<tr>
<td style="text-align: center;"><code>sys</code></td>
<td style="text-align: center;">3.4</td>
<td style="text-align: left;">Powerful and Reliable Tools for Running System Commands in R</td>
</tr>
<tr>
<td style="text-align: center;"><code>systemfit</code></td>
<td style="text-align: center;">1.1-24</td>
<td style="text-align: left;">Estimating Systems of Simultaneous Equations</td>
</tr>
<tr>
<td style="text-align: center;"><code>TAM</code></td>
<td style="text-align: center;">4.0-16</td>
<td style="text-align: left;">Test Analysis Modules</td>
</tr>
<tr>
<td style="text-align: center;"><code>tcltk</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">Tcl/Tk Interface</td>
</tr>
<tr>
<td style="text-align: center;"><code>tensor</code></td>
<td style="text-align: center;">1.5</td>
<td style="text-align: left;">Tensor product of arrays</td>
</tr>
<tr>
<td style="text-align: center;"><code>tensorA</code></td>
<td style="text-align: center;">0.36.2</td>
<td style="text-align: left;">Advanced Tensor Arithmetic with Named Indices</td>
</tr>
<tr>
<td style="text-align: center;"><code>terra</code></td>
<td style="text-align: center;">1.5-21</td>
<td style="text-align: left;">Spatial Data Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>testthat</code></td>
<td style="text-align: center;">3.1.4</td>
<td style="text-align: left;">Unit Testing for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>tgp</code></td>
<td style="text-align: center;">2.4-18</td>
<td style="text-align: left;">Bayesian Treed Gaussian Process Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>TH.data</code></td>
<td style="text-align: center;">1.1-1</td>
<td style="text-align: left;">TH's Data Archive</td>
</tr>
<tr>
<td style="text-align: center;"><code>threejs</code></td>
<td style="text-align: center;">0.3.3</td>
<td style="text-align: left;">Interactive 3D Scatter Plots, Networks and Globes</td>
</tr>
<tr>
<td style="text-align: center;"><code>tibble</code></td>
<td style="text-align: center;">3.1.7</td>
<td style="text-align: left;">Simple Data Frames</td>
</tr>
<tr>
<td style="text-align: center;"><code>tictoc</code></td>
<td style="text-align: center;">1.0.1</td>
<td style="text-align: left;">Functions for Timing R Scripts, as Well as Implementations of Stack and List Structures</td>
</tr>
<tr>
<td style="text-align: center;"><code>tidyr</code></td>
<td style="text-align: center;">1.2.0</td>
<td style="text-align: left;">Tidy Messy Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>tidyselect</code></td>
<td style="text-align: center;">1.1.2</td>
<td style="text-align: left;">Select from a Set of Strings</td>
</tr>
<tr>
<td style="text-align: center;"><code>tidyverse</code></td>
<td style="text-align: center;">1.3.1</td>
<td style="text-align: left;">Easily Install and Load the 'Tidyverse'</td>
</tr>
<tr>
<td style="text-align: center;"><code>timeDate</code></td>
<td style="text-align: center;">3043.102</td>
<td style="text-align: left;">Rmetrics - Chronological and Calendar Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>tinytex</code></td>
<td style="text-align: center;">0.39</td>
<td style="text-align: left;">Helper Functions to Install and Maintain TeX Live, and Compile LaTeX Documents</td>
</tr>
<tr>
<td style="text-align: center;"><code>tkrplot</code></td>
<td style="text-align: center;">0.0-26</td>
<td style="text-align: left;">TK Rplot</td>
</tr>
<tr>
<td style="text-align: center;"><code>tm</code></td>
<td style="text-align: center;">0.7-8</td>
<td style="text-align: left;">Text Mining Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>tmap</code></td>
<td style="text-align: center;">3.3-3</td>
<td style="text-align: left;">Thematic Maps</td>
</tr>
<tr>
<td style="text-align: center;"><code>tmaptools</code></td>
<td style="text-align: center;">3.1-1</td>
<td style="text-align: left;">Thematic Map Tools</td>
</tr>
<tr>
<td style="text-align: center;"><code>tmvnsim</code></td>
<td style="text-align: center;">1.0-2</td>
<td style="text-align: left;">Truncated Multivariate Normal Simulation</td>
</tr>
<tr>
<td style="text-align: center;"><code>tools</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">Tools for Package Development</td>
</tr>
<tr>
<td style="text-align: center;"><code>topicmodels</code></td>
<td style="text-align: center;">0.2-12</td>
<td style="text-align: left;">Topic Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>triebeard</code></td>
<td style="text-align: center;">0.3.0</td>
<td style="text-align: left;">'Radix' Trees in 'Rcpp'</td>
</tr>
<tr>
<td style="text-align: center;"><code>truncnorm</code></td>
<td style="text-align: center;">1.0-8</td>
<td style="text-align: left;">Truncated Normal Distribution</td>
</tr>
<tr>
<td style="text-align: center;"><code>trust</code></td>
<td style="text-align: center;">0.1-8</td>
<td style="text-align: left;">Trust Region Optimization</td>
</tr>
<tr>
<td style="text-align: center;"><code>tseries</code></td>
<td style="text-align: center;">0.10-51</td>
<td style="text-align: left;">Time Series Analysis and Computational Finance</td>
</tr>
<tr>
<td style="text-align: center;"><code>TTR</code></td>
<td style="text-align: center;">0.24.3</td>
<td style="text-align: left;">Technical Trading Rules</td>
</tr>
<tr>
<td style="text-align: center;"><code>tweenr</code></td>
<td style="text-align: center;">1.0.2</td>
<td style="text-align: left;">Interpolate Data for Smooth Animations</td>
</tr>
<tr>
<td style="text-align: center;"><code>TxDb.Hsapiens.UCSC.hg19.knownGene</code></td>
<td style="text-align: center;">3.2.2</td>
<td style="text-align: left;">Annotation package for TxDb object(s)</td>
</tr>
<tr>
<td style="text-align: center;"><code>tzdb</code></td>
<td style="text-align: center;">0.3.0</td>
<td style="text-align: left;">Time Zone Database Information</td>
</tr>
<tr>
<td style="text-align: center;"><code>ucminf</code></td>
<td style="text-align: center;">1.1-4.1</td>
<td style="text-align: left;">General-Purpose Unconstrained Non-Linear Optimization</td>
</tr>
<tr>
<td style="text-align: center;"><code>udunits2</code></td>
<td style="text-align: center;">0.13.2.1</td>
<td style="text-align: left;">Udunits-2 Bindings for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>units</code></td>
<td style="text-align: center;">0.8-0</td>
<td style="text-align: left;">Measurement Units for R Vectors</td>
</tr>
<tr>
<td style="text-align: center;"><code>UpSetR</code></td>
<td style="text-align: center;">1.4.0</td>
<td style="text-align: left;">A More Scalable Alternative to Venn and Euler Diagrams for Visualizing Intersecting Sets</td>
</tr>
<tr>
<td style="text-align: center;"><code>urca</code></td>
<td style="text-align: center;">1.3-0</td>
<td style="text-align: left;">Unit Root and Cointegration Tests for Time Series Data</td>
</tr>
<tr>
<td style="text-align: center;"><code>urltools</code></td>
<td style="text-align: center;">1.7.3</td>
<td style="text-align: left;">Vectorised Tools for URL Handling and Parsing</td>
</tr>
<tr>
<td style="text-align: center;"><code>usethis</code></td>
<td style="text-align: center;">2.1.6</td>
<td style="text-align: left;">Automate Package and Project Setup</td>
</tr>
<tr>
<td style="text-align: center;"><code>utf8</code></td>
<td style="text-align: center;">1.2.2</td>
<td style="text-align: left;">Unicode Text Processing</td>
</tr>
<tr>
<td style="text-align: center;"><code>utils</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">The R Utils Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>uuid</code></td>
<td style="text-align: center;">1.1-0</td>
<td style="text-align: left;">Tools for Generating and Handling of UUIDs</td>
</tr>
<tr>
<td style="text-align: center;"><code>V8</code></td>
<td style="text-align: center;">4.2.0</td>
<td style="text-align: left;">Embedded JavaScript and WebAssembly Engine for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>VariantAnnotation</code></td>
<td style="text-align: center;">1.42.1</td>
<td style="text-align: left;">Annotation of Genetic Variants</td>
</tr>
<tr>
<td style="text-align: center;"><code>vctrs</code></td>
<td style="text-align: center;">0.4.1</td>
<td style="text-align: left;">Vector Helpers</td>
</tr>
<tr>
<td style="text-align: center;"><code>vegan</code></td>
<td style="text-align: center;">2.6-2</td>
<td style="text-align: left;">Community Ecology Package</td>
</tr>
<tr>
<td style="text-align: center;"><code>VGAM</code></td>
<td style="text-align: center;">1.1-6</td>
<td style="text-align: left;">Vector Generalized Linear and Additive Models</td>
</tr>
<tr>
<td style="text-align: center;"><code>VineCopula</code></td>
<td style="text-align: center;">2.4.4</td>
<td style="text-align: left;">Statistical Inference of Vine Copulas</td>
</tr>
<tr>
<td style="text-align: center;"><code>viridis</code></td>
<td style="text-align: center;">0.6.2</td>
<td style="text-align: left;">Colorblind-Friendly Color Maps for R</td>
</tr>
<tr>
<td style="text-align: center;"><code>viridisLite</code></td>
<td style="text-align: center;">0.4.0</td>
<td style="text-align: left;">Colorblind-Friendly Color Maps (Lite Version)</td>
</tr>
<tr>
<td style="text-align: center;"><code>vpc</code></td>
<td style="text-align: center;">1.2.2</td>
<td style="text-align: left;">Create Visual Predictive Checks</td>
</tr>
<tr>
<td style="text-align: center;"><code>vroom</code></td>
<td style="text-align: center;">1.5.7</td>
<td style="text-align: left;">Read and Write Rectangular Text Data Quickly</td>
</tr>
<tr>
<td style="text-align: center;"><code>vsn</code></td>
<td style="text-align: center;">3.64.0</td>
<td style="text-align: left;">Variance stabilization and calibration for microarray data</td>
</tr>
<tr>
<td style="text-align: center;"><code>waldo</code></td>
<td style="text-align: center;">0.4.0</td>
<td style="text-align: left;">Find Differences Between R Objects</td>
</tr>
<tr>
<td style="text-align: center;"><code>wateRmelon</code></td>
<td style="text-align: center;">2.2.0</td>
<td style="text-align: left;">Illumina 450 and EPIC methylation array normalization and metrics</td>
</tr>
<tr>
<td style="text-align: center;"><code>WGCNA</code></td>
<td style="text-align: center;">1.71</td>
<td style="text-align: left;">Weighted Correlation Network Analysis</td>
</tr>
<tr>
<td style="text-align: center;"><code>whisker</code></td>
<td style="text-align: center;">0.4</td>
<td style="text-align: left;">{{mustache}} for R, Logicless Templating</td>
</tr>
<tr>
<td style="text-align: center;"><code>widgetframe</code></td>
<td style="text-align: center;">0.3.1</td>
<td style="text-align: left;">'Htmlwidgets' in Responsive 'iframes'</td>
</tr>
<tr>
<td style="text-align: center;"><code>widgetTools</code></td>
<td style="text-align: center;">1.74.0</td>
<td style="text-align: left;">Creates an interactive tcltk widget</td>
</tr>
<tr>
<td style="text-align: center;"><code>withr</code></td>
<td style="text-align: center;">2.5.0</td>
<td style="text-align: left;">Run Code 'With' Temporarily Modified Global State</td>
</tr>
<tr>
<td style="text-align: center;"><code>wk</code></td>
<td style="text-align: center;">0.6.0</td>
<td style="text-align: left;">Lightweight Well-Known Geometry Parsing</td>
</tr>
<tr>
<td style="text-align: center;"><code>xfun</code></td>
<td style="text-align: center;">0.31</td>
<td style="text-align: left;">Supporting Functions for Packages Maintained by 'Yihui Xie'</td>
</tr>
<tr>
<td style="text-align: center;"><code>xgxr</code></td>
<td style="text-align: center;">1.1.1</td>
<td style="text-align: left;">Exploratory Graphics for Pharmacometrics</td>
</tr>
<tr>
<td style="text-align: center;"><code>XML</code></td>
<td style="text-align: center;">3.99-0.9</td>
<td style="text-align: left;">Tools for Parsing and Generating XML Within R and S-Plus</td>
</tr>
<tr>
<td style="text-align: center;"><code>xml2</code></td>
<td style="text-align: center;">1.3.3</td>
<td style="text-align: left;">Parse XML</td>
</tr>
<tr>
<td style="text-align: center;"><code>xopen</code></td>
<td style="text-align: center;">1.0.0</td>
<td style="text-align: left;">Open System Files, 'URLs', Anything</td>
</tr>
<tr>
<td style="text-align: center;"><code>xtable</code></td>
<td style="text-align: center;">1.8-4</td>
<td style="text-align: left;">Export Tables to LaTeX or HTML</td>
</tr>
<tr>
<td style="text-align: center;"><code>xts</code></td>
<td style="text-align: center;">0.12.1</td>
<td style="text-align: left;">eXtensible Time Series</td>
</tr>
<tr>
<td style="text-align: center;"><code>XVector</code></td>
<td style="text-align: center;">0.36.0</td>
<td style="text-align: left;">Foundation of external vector representation and manipulation in Bioconductor</td>
</tr>
<tr>
<td style="text-align: center;"><code>yaml</code></td>
<td style="text-align: center;">2.3.5</td>
<td style="text-align: left;">Methods to Convert R Data to YAML and Back</td>
</tr>
<tr>
<td style="text-align: center;"><code>zip</code></td>
<td style="text-align: center;">2.2.0</td>
<td style="text-align: left;">Cross-Platform 'zip' Compression</td>
</tr>
<tr>
<td style="text-align: center;"><code>zlibbioc</code></td>
<td style="text-align: center;">1.42.0</td>
<td style="text-align: left;">An R packaged zlib-1.2.5</td>
</tr>
<tr>
<td style="text-align: center;"><code>zoo</code></td>
<td style="text-align: center;">1.8-10</td>
<td style="text-align: left;">S3 Infrastructure for Regular and Irregular Time Series (Z's Ordered Observations)</td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2014 - 2024 UCL Research Computing Group
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://github.com/UCL-ARC" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6m-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3m44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9M244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8M97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1m-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7m32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1m-11.4-14.7c-1.6 1-1.6 3.6 0 5.9s4.3 3.3 5.6 2.3c1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "../..", "features": ["instant"], "search": "../../assets/javascripts/workers/search.f8cc74c7.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.c8b220af.min.js"></script>
      
    
  </body>
</html>